--- a/net/minecraft/server/MinecraftServer.java	2023-03-30 19:29:27
+++ b/net/minecraft/server/MinecraftServer.java	2023-03-30 20:05:22
@@ -619,7 +619,7 @@
          }
 
          this.timeReference = Util.getMeasuringTimeMs();
-         this.favicon = (ServerMetadata.Favicon)this.loadFavicon().orElse((Object)null);
+         this.favicon = (ServerMetadata.Favicon)this.loadFavicon().orElse(null);
          this.metadata = this.createMetadata();
 
          while(this.running) {
@@ -740,7 +740,7 @@
       }
    }
 
-   protected void executeTask(ServerTask serverTask) {
+   public void executeTask(ServerTask serverTask) {
       this.getProfiler().visit("runTask");
       super.executeTask(serverTask);
    }
@@ -1809,8 +1809,6 @@
    }
 
    static record ResourceManagerHolder(LifecycledResourceManager resourceManager, DataPackContents dataPackContents) implements AutoCloseable {
-      final LifecycledResourceManager resourceManager;
-      final DataPackContents dataPackContents;
 
       public void close() {
          this.resourceManager.close();
@@ -1819,4 +1817,4 @@
 
    public static record ServerResourcePackProperties(String url, String hash, boolean isRequired, @Nullable Text prompt) {
    }
-}
+}
\ No newline at end of file
