--- a/net/minecraft/server/PlayerManager.java
+++ b/net/minecraft/server/PlayerManager.java
@@ -334,7 +334,7 @@
          if (entity.hasPlayerRider()) {
             LOGGER.debug("Removing player mount");
             player.stopRiding();
-            entity.streamPassengersAndSelf().forEach((entity) -> entity.setRemoved(Entity.RemovalReason.UNLOADED_WITH_PLAYER));
+            entity.streamPassengersAndSelf().forEach((entity2) -> entity2.setRemoved(Entity.RemovalReason.UNLOADED_WITH_PLAYER));
          }
       }
 
