--- a/net/minecraft/server/SaveLoading.java	1970-01-12 08:58:04
+++ b/net/minecraft/server/SaveLoading.java	2023-03-30 19:23:38
@@ -75,8 +75,6 @@
    }
 
    public static record LoadContext<D>(D extraData, DynamicRegistryManager.Immutable dimensionsRegistryManager) {
-      final D extraData;
-      final DynamicRegistryManager.Immutable dimensionsRegistryManager;
    }
 
    @FunctionalInterface
@@ -93,6 +91,5 @@
    }
 
    public static record ServerConfig(SaveLoading.DataPacks dataPacks, CommandManager.RegistrationEnvironment commandEnvironment, int functionPermissionLevel) {
-      final SaveLoading.DataPacks dataPacks;
    }
-}
+}
\ No newline at end of file
