--- a/net/minecraft/world/chunk/PalettedContainer.java	1970-01-12 08:58:04
+++ b/net/minecraft/world/chunk/PalettedContainer.java	2023-03-30 19:23:30
@@ -276,8 +276,6 @@
    }
 
    static record Data<T>(PalettedContainer.DataProvider<T> configuration, PaletteStorage storage, Palette<T> palette) {
-      final PaletteStorage storage;
-      final Palette<T> palette;
 
       public void importFrom(Palette<T> palette, PaletteStorage storage) {
          for(int i = 0; i < storage.getSize(); ++i) {
@@ -382,4 +380,4 @@
          return dataProvider.factory() == ID_LIST ? i : dataProvider.bits();
       }
    }
-}
+}
\ No newline at end of file
