--- a/net/minecraft/world/chunk/WorldChunk.java
+++ b/net/minecraft/world/chunk/WorldChunk.java
@@ -590,7 +590,7 @@
 
    private <T extends BlockEntity> void updateTicker(T blockEntity) {
       BlockState blockState = blockEntity.getCachedState();
-      BlockEntityTicker<T> blockEntityTicker = blockState.getBlockEntityTicker(this.world, blockEntity.getType());
+      BlockEntityTicker<T> blockEntityTicker = (BlockEntityTicker<T>) blockState.getBlockEntityTicker(this.world, blockEntity.getType());
       if (blockEntityTicker == null) {
          this.removeBlockEntityTicker(blockEntity.getPos());
       } else {
