--- a/java/net/minecraft/block/entity/SculkSensorBlockEntity.java
+++ b/java/net/minecraft/block/entity/SculkSensorBlockEntity.java
@@ -31,7 +31,7 @@
       super.readNbt(nbt);
       this.lastVibrationFrequency = nbt.getInt("last_vibration_frequency");
       if (nbt.contains("listener", 10)) {
-         VibrationListener.createCodec(this).parse(new Dynamic(NbtOps.INSTANCE, nbt.getCompound("listener"))).resultOrPartial(LOGGER::error).ifPresent((listener) -> {
+         VibrationListener.createCodec(this).parse(new Dynamic<>(NbtOps.INSTANCE, nbt.getCompound("listener"))).resultOrPartial(LOGGER::error).ifPresent((listener) -> {
             this.listener = listener;
          });
       }
