--- a/net/minecraft/entity/ai/brain/task/OpenDoorsTask.java
+++ b/net/minecraft/entity/ai/brain/task/OpenDoorsTask.java
@@ -29,7 +29,7 @@
    private static final double REACH_DISTANCE = 2.0D;
 
    public static Task<LivingEntity> create() {
-      MutableObject<PathNode> mutableObject = new MutableObject((Object)null);
+      MutableObject<PathNode> mutableObject = new MutableObject(null);
       MutableInt mutableInt = new MutableInt(0);
       return TaskTriggerer.task((context) -> context.group(context.queryMemoryValue(MemoryModuleType.PATH), context.queryMemoryOptional(MemoryModuleType.DOORS_TO_CLOSE), context.queryMemoryOptional(MemoryModuleType.MOBS)).apply(context, (path, doorsToClose, mobs) -> (world, entity, time) -> {
                Path path = context.getValue(path);
