--- a/net/minecraft/entity/ai/brain/task/RidingTask.java
+++ b/net/minecraft/entity/ai/brain/task/RidingTask.java
@@ -9,7 +9,7 @@
    public static <E extends LivingEntity> Task<E> create(int range, BiPredicate<E, Entity> alternativeRideCondition) {
       return TaskTriggerer.task((context) -> context.group(context.queryMemoryOptional(MemoryModuleType.RIDE_TARGET)).apply(context, (rideTarget) -> (world, entity, time) -> {
                Entity entity2 = entity.getVehicle();
-               Entity entity3 = (Entity)context.getOptionalValue(rideTarget).orElse((Object)null);
+               Entity entity3 = (Entity)context.getOptionalValue(rideTarget).orElse(null);
                if (entity2 == null && entity3 == null) {
                   return false;
                } else {
