--- a/net/minecraft/client/gui/screen/world/WorldListWidget.java
+++ b/net/minecraft/client/gui/screen/world/WorldListWidget.java
@@ -95,7 +95,7 @@
    @Nullable
    private List<LevelSummary> tryGet() {
       try {
-         return (List)this.levelsFuture.getNow((Object)null);
+         return (List)this.levelsFuture.getNow(null);
       } catch (CancellationException | CompletionException var2) {
          return null;
       }
