--- a/net/minecraft/client/gui/screen/world/WorldCreator.java
+++ b/net/minecraft/client/gui/screen/world/WorldCreator.java
@@ -51,7 +51,7 @@
    public WorldCreator(Path savesDirectory, GeneratorOptionsHolder generatorOptionsHolder, Optional<RegistryKey<WorldPreset>> defaultWorldType, OptionalLong seed) {
       this.savesDirectory = savesDirectory;
       this.generatorOptionsHolder = generatorOptionsHolder;
-      this.worldType = new WorldCreator.WorldType((RegistryEntry)getWorldPreset(generatorOptionsHolder, defaultWorldType).orElse((Object)null));
+      this.worldType = new WorldCreator.WorldType((RegistryEntry)getWorldPreset(generatorOptionsHolder, defaultWorldType).orElse(null));
       this.updateWorldTypeLists();
       this.seed = seed.isPresent() ? Long.toString(seed.getAsLong()) : "";
       this.generateStructures = generatorOptionsHolder.generatorOptions().shouldGenerateStructures();
