--- a/java/net/minecraft/util/math/noise/DoublePerlinNoiseSampler.java
+++ b/java/net/minecraft/util/math/noise/DoublePerlinNoiseSampler.java
@@ -95,8 +95,6 @@
    }
 
    public static record NoiseParameters(int firstOctave, DoubleList amplitudes) {
-      final int firstOctave;
-      final DoubleList amplitudes;
       public static final Codec<DoublePerlinNoiseSampler.NoiseParameters> CODEC = RecordCodecBuilder.create((instance) -> {
          return instance.group(Codec.INT.fieldOf("firstOctave").forGetter(DoublePerlinNoiseSampler.NoiseParameters::firstOctave), Codec.DOUBLE.listOf().fieldOf("amplitudes").forGetter(DoublePerlinNoiseSampler.NoiseParameters::amplitudes)).apply(instance, DoublePerlinNoiseSampler.NoiseParameters::new);
       });
@@ -112,4 +110,4 @@
          }));
       }
    }
 }
