--- a/net/minecraft/network/packet/s2c/play/EntityStatusEffectS2CPacket.java	2023-03-30 19:29:25
+++ b/net/minecraft/network/packet/s2c/play/EntityStatusEffectS2CPacket.java	2023-03-30 19:29:45
@@ -40,7 +40,7 @@
       }
 
       this.flags = b;
-      this.factorCalculationData = (StatusEffectInstance.FactorCalculationData)effect.getFactorCalculationData().orElse((Object)null);
+      this.factorCalculationData = (StatusEffectInstance.FactorCalculationData)effect.getFactorCalculationData().orElse(null);
    }
 
    public EntityStatusEffectS2CPacket(PacketByteBuf buf) {
