--- a/net/minecraft/network/packet/s2c/play/PlayerListS2CPacket.java	1970-01-12 08:58:04
+++ b/net/minecraft/network/packet/s2c/play/PlayerListS2CPacket.java	2023-03-30 19:23:16
@@ -114,7 +114,6 @@
 
    public static record Entry(UUID profileId, GameProfile profile, boolean listed, int latency, GameMode gameMode, @Nullable Text displayName, @Nullable PublicPlayerSession.Serialized chatSession) {
       @Nullable
-      final PublicPlayerSession.Serialized chatSession;
 
       Entry(ServerPlayerEntity player) {
          this(player.getUuid(), player.getGameProfile(), true, player.pingMilliseconds, player.interactionManager.getGameMode(), player.getPlayerListName(), Nullables.map(player.getSession(), PublicPlayerSession::toSerialized));
@@ -141,4 +140,4 @@
          return new PlayerListS2CPacket.Entry(this.profileId, this.gameProfile, this.listed, this.latency, this.gameMode, this.displayName, this.session);
       }
    }
-}
+}
\ No newline at end of file
