--- a/net/minecraft/client/render/entity/feature/HorseMarkingFeatureRenderer.java	2023-03-30 19:29:28
+++ b/net/minecraft/client/render/entity/feature/HorseMarkingFeatureRenderer.java	2023-03-30 19:29:45
@@ -18,7 +18,7 @@
 @Environment(EnvType.CLIENT)
 public class HorseMarkingFeatureRenderer extends FeatureRenderer<HorseEntity, HorseEntityModel<HorseEntity>> {
    private static final Map<HorseMarking, Identifier> TEXTURES = Util.make(Maps.newEnumMap(HorseMarking.class), (textures) -> {
-      textures.put(HorseMarking.NONE, (Object)null);
+      textures.put(HorseMarking.NONE, null);
       textures.put(HorseMarking.WHITE, new Identifier("textures/entity/horse/horse_markings_white.png"));
       textures.put(HorseMarking.WHITE_FIELD, new Identifier("textures/entity/horse/horse_markings_whitefield.png"));
       textures.put(HorseMarking.WHITE_DOTS, new Identifier("textures/entity/horse/horse_markings_whitedots.png"));
