--- a/net/minecraft/world/dimension/DimensionType.java	1970-01-12 08:58:04
+++ b/net/minecraft/world/dimension/DimensionType.java	2023-03-30 20:05:23
@@ -35,15 +35,15 @@
    public static final Codec<RegistryEntry<DimensionType>> REGISTRY_CODEC = RegistryElementCodec.of(RegistryKeys.DIMENSION_TYPE, CODEC);
 
    public DimensionType {
-      if (j < 16) {
+      if (height < 16) {
          throw new IllegalStateException("height has to be at least 16");
-      } else if (i + j > MAX_COLUMN_HEIGHT + 1) {
+      } else if (minY + height > MAX_COLUMN_HEIGHT + 1) {
          throw new IllegalStateException("min_y + height cannot be higher than: " + (MAX_COLUMN_HEIGHT + 1));
-      } else if (k > j) {
+      } else if (logicalHeight > height) {
          throw new IllegalStateException("logical_height cannot be higher than height");
-      } else if (j % 16 != 0) {
+      } else if (height % 16 != 0) {
          throw new IllegalStateException("height has to be multiple of 16");
-      } else if (i % 16 != 0) {
+      } else if (minY % 16 != 0) {
          throw new IllegalStateException("min_y has to be a multiple of 16");
       }
    }
