--- a/java/net/minecraft/registry/DynamicRegistryManager.java
+++ b/java/net/minecraft/registry/DynamicRegistryManager.java
@@ -31,8 +31,8 @@
    static DynamicRegistryManager.Immutable of(final Registry<? extends Registry<?>> registries) {
       return new DynamicRegistryManager.Immutable() {
          public <T> Optional<Registry<T>> getOptional(RegistryKey<? extends Registry<? extends T>> key) {
-            Registry<Registry<T>> registry = registries;
-            return registry.getOrEmpty(key);
+            Registry<Registry<T>> registry = (Registry<Registry<T>>) registries;
+            return registry.getOrEmpty((RegistryKey<Registry<T>>) key);
          }
 
          public Stream<DynamicRegistryManager.Entry<?>> streamAllRegistries() {
@@ -62,14 +62,13 @@
    }
 
    public static record Entry<T>(RegistryKey<? extends Registry<T>> key, Registry<T> value) {
-      final Registry<T> value;
 
       private static <T, R extends Registry<? extends T>> DynamicRegistryManager.Entry<T> of(Map.Entry<? extends RegistryKey<? extends Registry<?>>, R> entry) {
          return of(entry.getKey(), entry.getValue());
       }
 
       private static <T> DynamicRegistryManager.Entry<T> of(RegistryKey<? extends Registry<?>> key, Registry<?> value) {
-         return new DynamicRegistryManager.Entry<>(key, value);
+         return new DynamicRegistryManager.Entry(key, value);
       }
 
       private DynamicRegistryManager.Entry<T> freeze() {
@@ -99,7 +98,7 @@
 
       public <E> Optional<Registry<E>> getOptional(RegistryKey<? extends Registry<? extends E>> key) {
          return Optional.ofNullable(this.registries.get(key)).map((registry) -> {
-            return registry;
+            return (Registry<E>) registry;
          });
       }
 
