--- a/net/minecraft/entity/mob/AbstractPiglinEntity.java
+++ b/net/minecraft/entity/mob/AbstractPiglinEntity.java
@@ -114,7 +114,7 @@
 
    @Nullable
    public LivingEntity getTarget() {
-      return (LivingEntity)this.brain.getOptionalRegisteredMemory(MemoryModuleType.ATTACK_TARGET).orElse((Object)null);
+      return (LivingEntity)this.brain.getOptionalRegisteredMemory(MemoryModuleType.ATTACK_TARGET).orElse(null);
    }
 
    protected boolean isHoldingTool() {
