--- a/net/minecraft/entity/mob/DrownedEntity.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/mob/DrownedEntity.java	2023-03-30 20:31:32
@@ -75,11 +75,11 @@
       this.goalSelector.add(6, new DrownedEntity.TargetAboveWaterGoal(this, 1.0D, this.world.getSeaLevel()));
       this.goalSelector.add(7, new WanderAroundGoal(this, 1.0D));
       this.targetSelector.add(1, (new RevengeGoal(this, DrownedEntity.class)).setGroupRevenge(ZombifiedPiglinEntity.class));
-      this.targetSelector.add(2, new ActiveTargetGoal(this, PlayerEntity.class, 10, true, false, this::canDrownedAttackTarget));
-      this.targetSelector.add(3, new ActiveTargetGoal(this, MerchantEntity.class, false));
-      this.targetSelector.add(3, new ActiveTargetGoal(this, IronGolemEntity.class, true));
-      this.targetSelector.add(3, new ActiveTargetGoal(this, AxolotlEntity.class, true, false));
-      this.targetSelector.add(5, new ActiveTargetGoal(this, TurtleEntity.class, 10, true, false, TurtleEntity.BABY_TURTLE_ON_LAND_FILTER));
+      this.targetSelector.add(2, new ActiveTargetGoal<>(this, PlayerEntity.class, 10, true, false, this::canDrownedAttackTarget));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, MerchantEntity.class, false));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, IronGolemEntity.class, true));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, AxolotlEntity.class, true, false));
+      this.targetSelector.add(5, new ActiveTargetGoal<>(this, TurtleEntity.class, 10, true, false, TurtleEntity.BABY_TURTLE_ON_LAND_FILTER));
    }
 
    public EntityData initialize(ServerWorldAccess world, LocalDifficulty difficulty, SpawnReason spawnReason, @Nullable EntityData entityData, @Nullable NbtCompound entityNbt) {
