--- a/net/minecraft/entity/mob/EndermanEntity.java	2023-03-30 19:29:28
+++ b/net/minecraft/entity/mob/EndermanEntity.java	2023-03-30 19:29:44
@@ -326,7 +326,7 @@
 
    @Nullable
    public BlockState getCarriedBlock() {
-      return (BlockState)((Optional)this.dataTracker.get(CARRIED_BLOCK)).orElse((Object)null);
+      return (BlockState)((Optional)this.dataTracker.get(CARRIED_BLOCK)).orElse(null);
    }
 
    public boolean damage(DamageSource source, float amount) {
