--- a/net/minecraft/entity/mob/MobEntity.java
+++ b/net/minecraft/entity/mob/MobEntity.java
@@ -1108,7 +1108,7 @@
          if (itemStack.getItem() instanceof SpawnEggItem) {
             if (this.world instanceof ServerWorld) {
                SpawnEggItem spawnEggItem = (SpawnEggItem)itemStack.getItem();
-               Optional<MobEntity> optional = spawnEggItem.spawnBaby(player, this, this.getType(), (ServerWorld)this.world, this.getPos(), itemStack);
+               Optional<MobEntity> optional = spawnEggItem.spawnBaby(player, this, (EntityType<? extends MobEntity>) this.getType(), (ServerWorld)this.world, this.getPos(), itemStack);
                optional.ifPresent((entity) -> this.onPlayerSpawnedChild(player, entity));
                return optional.isPresent() ? ActionResult.SUCCESS : ActionResult.PASS;
             } else {
