--- a/net/minecraft/entity/mob/PiglinBrain.java	2023-03-30 19:29:24
+++ b/net/minecraft/entity/mob/PiglinBrain.java	2023-03-30 19:29:44
@@ -187,9 +187,9 @@
 
    protected static void tickActivities(PiglinEntity piglin) {
       Brain<PiglinEntity> brain = piglin.getBrain();
-      Activity activity = (Activity)brain.getFirstPossibleNonCoreActivity().orElse((Object)null);
+      Activity activity = (Activity)brain.getFirstPossibleNonCoreActivity().orElse(null);
       brain.resetPossibleActivities(ImmutableList.of(Activity.ADMIRE_ITEM, Activity.FIGHT, Activity.AVOID, Activity.CELEBRATE, Activity.RIDE, Activity.IDLE));
-      Activity activity2 = (Activity)brain.getFirstPossibleNonCoreActivity().orElse((Object)null);
+      Activity activity2 = (Activity)brain.getFirstPossibleNonCoreActivity().orElse(null);
       if (activity != activity2) {
          getCurrentActivitySound(piglin).ifPresent(piglin::playSound);
       }
