--- a/net/minecraft/entity/mob/PiglinEntity.java
+++ b/net/minecraft/entity/mob/PiglinEntity.java
@@ -202,7 +202,7 @@
    }
 
    public Brain<PiglinEntity> getBrain() {
-      return super.getBrain();
+      return (Brain<PiglinEntity>) super.getBrain();
    }
 
    public ActionResult interactMob(PlayerEntity player, Hand hand) {
@@ -390,7 +390,7 @@
    }
 
    protected SoundEvent getAmbientSound() {
-      return this.world.isClient ? null : (SoundEvent)PiglinBrain.getCurrentActivitySound(this).orElse((Object)null);
+      return this.world.isClient ? null : (SoundEvent)PiglinBrain.getCurrentActivitySound(this).orElse(null);
    }
 
    protected SoundEvent getHurtSound(DamageSource source) {
