--- a/net/minecraft/entity/mob/RavagerEntity.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/mob/RavagerEntity.java	2023-03-30 20:35:47
@@ -67,9 +67,9 @@
       this.goalSelector.add(6, new LookAtEntityGoal(this, PlayerEntity.class, 6.0F));
       this.goalSelector.add(10, new LookAtEntityGoal(this, MobEntity.class, 8.0F));
       this.targetSelector.add(2, (new RevengeGoal(this, RaiderEntity.class)).setGroupRevenge());
-      this.targetSelector.add(3, new ActiveTargetGoal(this, PlayerEntity.class, true));
-      this.targetSelector.add(4, new ActiveTargetGoal(this, MerchantEntity.class, true, (entity) -> !entity.isBaby()));
-      this.targetSelector.add(4, new ActiveTargetGoal(this, IronGolemEntity.class, true));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, PlayerEntity.class, true));
+      this.targetSelector.add(4, new ActiveTargetGoal<>(this, MerchantEntity.class, true, (entity) -> !entity.isBaby()));
+      this.targetSelector.add(4, new ActiveTargetGoal<>(this, IronGolemEntity.class, true));
    }
 
    protected void updateGoalControls() {
