--- a/net/minecraft/entity/mob/SpellcastingIllagerEntity.java
+++ b/net/minecraft/entity/mob/SpellcastingIllagerEntity.java
@@ -189,7 +189,7 @@
       DISAPPEAR(4, 0.3D, 0.3D, 0.8D),
       BLINDNESS(5, 0.1D, 0.1D, 0.2D);
 
-      private static final IntFunction<SpellcastingIllagerEntity.Spell> BY_ID = ValueLists.createIdToValueFunction((spell) -> spell.id, values(), ValueLists.OutOfBoundsHandling.ZERO);
+      private static final IntFunction<SpellcastingIllagerEntity.Spell> BY_ID = ValueLists.<SpellcastingIllagerEntity.Spell>createIdToValueFunction((spell) -> spell.id, values(), ValueLists.OutOfBoundsHandling.ZERO);
       final int id;
       final double[] particleVelocity;
 
