--- a/net/minecraft/scoreboard/Scoreboard.java
+++ b/net/minecraft/scoreboard/Scoreboard.java
@@ -60,7 +60,7 @@
    }
 
    public final void forEachScore(ScoreboardCriterion criterion, String player, Consumer<ScoreboardPlayerScore> action) {
-      ((List)this.objectivesByCriterion.getOrDefault(criterion, Collections.emptyList())).forEach((objective) -> action.accept(this.getPlayerScore(player, objective)));
+      (this.objectivesByCriterion.getOrDefault(criterion, Collections.emptyList())).forEach((objective) -> action.accept(this.getPlayerScore(player, objective)));
    }
 
    public boolean playerHasObjective(String playerName, ScoreboardObjective objective) {
