--- a/net/minecraft/world/event/listener/VibrationSelector.java
+++ b/net/minecraft/world/event/listener/VibrationSelector.java
@@ -49,7 +49,7 @@
       if (this.current.isEmpty()) {
          return Optional.empty();
       } else {
-         return ((Pair)this.current.get()).getRight() < currentTick ? Optional.of((Vibration)((Pair)this.current.get()).getLeft()) : Optional.empty();
+         return (this.current.get()).getRight() < currentTick ? Optional.of((Vibration)((Pair)this.current.get()).getLeft()) : Optional.empty();
       }
    }
 
