--- a/net/minecraft/network/message/LastSeenMessageList.java	1970-01-12 08:58:04
+++ b/net/minecraft/network/message/LastSeenMessageList.java	2023-03-30 20:05:23
@@ -43,7 +43,7 @@
       public static final LastSeenMessageList.Indexed EMPTY = new LastSeenMessageList.Indexed(List.of());
 
       public Indexed(PacketByteBuf buf) {
-         this(buf.readCollection(PacketByteBuf.getMaxValidator(ArrayList::new, 20), MessageSignatureData.Indexed::fromBuf));
+         this(buf.<MessageSignatureData.Indexed, List<MessageSignatureData.Indexed>>readCollection(PacketByteBuf.getMaxValidator(ArrayList::new, 20), MessageSignatureData.Indexed::fromBuf));
       }
 
       public void write(PacketByteBuf buf) {
