--- a/net/minecraft/client/realms/gui/screen/RealmsResetNormalWorldScreen.java	2023-03-30 19:29:27
+++ b/net/minecraft/client/realms/gui/screen/RealmsResetNormalWorldScreen.java	2023-03-30 19:29:45
@@ -42,7 +42,7 @@
    }
 
    public void close() {
-      this.callback.accept((Object)null);
+      this.callback.accept(null);
    }
 
    public void render(MatrixStack matrices, int mouseX, int mouseY, float delta) {
