--- a/net/minecraft/client/item/ModelPredicateProviderRegistry.java
+++ b/net/minecraft/client/item/ModelPredicateProviderRegistry.java
@@ -146,7 +146,7 @@
       register(Items.COMPASS, new Identifier("angle"), new CompassAnglePredicateProvider((world, stack, entity) -> CompassItem.hasLodestone(stack) ? CompassItem.createLodestonePos(stack.getOrCreateNbt()) : CompassItem.createSpawnPos(world)));
       register(Items.RECOVERY_COMPASS, new Identifier("angle"), new CompassAnglePredicateProvider((world, stack, entity) -> {
          if (entity instanceof PlayerEntity playerEntity) {
-            return (GlobalPos)playerEntity.getLastDeathPos().orElse((Object)null);
+            return (GlobalPos)playerEntity.getLastDeathPos().orElse(null);
          } else {
             return null;
          }
