--- a/net/minecraft/nbt/NbtByteArray.java	1970-01-12 08:58:04
+++ b/net/minecraft/nbt/NbtByteArray.java	2023-03-30 20:05:23
@@ -122,7 +122,7 @@
       return NbtByte.of(b);
    }
 
-   public void method_10531(int i, NbtByte nbtByte) {
+   public void add(int i, NbtByte nbtByte) {
       this.value = ArrayUtils.add(this.value, i, nbtByte.byteValue());
    }
 
@@ -144,7 +144,7 @@
       }
    }
 
-   public NbtByte method_10536(int i) {
+   public NbtByte remove(int i) {
       byte b = this.value[i];
       this.value = ArrayUtils.remove(this.value, i);
       return NbtByte.of(b);
