--- a/net/minecraft/client/render/debug/ChunkLoadingDebugRenderer.java
+++ b/net/minecraft/client/render/debug/ChunkLoadingDebugRenderer.java
@@ -46,7 +46,7 @@
       }
 
       if (this.loadingData != null) {
-         Map<ChunkPos, String> map = (Map)this.loadingData.serverStates.getNow((Object)null);
+         Map<ChunkPos, String> map = (Map)this.loadingData.serverStates.getNow(null);
          double e = this.client.gameRenderer.getCamera().getPos().y * 0.85D;
 
          for(Map.Entry<ChunkPos, String> entry : this.loadingData.clientStates.entrySet()) {
