--- a/net/minecraft/client/render/debug/GameEventDebugRenderer.java
+++ b/net/minecraft/client/render/debug/GameEventDebugRenderer.java
@@ -105,8 +105,6 @@
 
    @Environment(EnvType.CLIENT)
    static record Entry(long startingMs, GameEvent event, Vec3d pos) {
-      final GameEvent event;
-      final Vec3d pos;
 
       public boolean hasExpired() {
          return Util.getMeasuringTimeMs() - this.startingMs > 3000L;
@@ -143,4 +141,4 @@
          return false;
       }
    }
-}
+}
\ No newline at end of file
