--- a/net/minecraft/data/server/tag/TagProvider.java
+++ b/net/minecraft/data/server/tag/TagProvider.java
@@ -56,12 +56,10 @@
 
    public CompletableFuture<?> run(DataWriter writer) {
       return this.getRegistryLookupFuture().thenApply((registryLookupFuture) -> {
-         this.registryLoadFuture.complete((Object)null);
+         this.registryLoadFuture.complete(null);
          return registryLookupFuture;
       }).thenCombineAsync(this.parentTagLookupFuture, (lookup, parent) -> {
          record RegistryInfo<T>(RegistryWrapper.WrapperLookup contents, TagProvider.TagLookup<T> parent) {
-            final RegistryWrapper.WrapperLookup contents;
-            final TagProvider.TagLookup<T> parent;
          }
 
          return new RegistryInfo(lookup, parent);
@@ -153,4 +151,4 @@
          return ((Optional)this.apply(tag)).isPresent();
       }
    }
-}
+}
\ No newline at end of file
