--- a/net/minecraft/entity/passive/CamelEntity.java
+++ b/net/minecraft/entity/passive/CamelEntity.java
@@ -136,7 +136,7 @@
 
    protected void mobTick() {
       this.world.getProfiler().push("camelBrain");
-      Brain<?> brain = this.getBrain();
+      Brain<CamelEntity> brain = (Brain<CamelEntity>) this.getBrain();
       brain.tick((ServerWorld)this.world, this);
       this.world.getProfiler().pop();
       this.world.getProfiler().push("camelActivityUpdate");
