--- a/net/minecraft/entity/passive/IronGolemEntity.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/passive/IronGolemEntity.java	2023-03-30 20:33:47
@@ -77,8 +77,8 @@
       this.goalSelector.add(8, new LookAroundGoal(this));
       this.targetSelector.add(1, new TrackIronGolemTargetGoal(this));
       this.targetSelector.add(2, new RevengeGoal(this));
-      this.targetSelector.add(3, new ActiveTargetGoal(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
-      this.targetSelector.add(3, new ActiveTargetGoal(this, MobEntity.class, 5, false, false, (entity) -> entity instanceof Monster && !(entity instanceof CreeperEntity)));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, MobEntity.class, 5, false, false, (entity) -> entity instanceof Monster && !(entity instanceof CreeperEntity)));
       this.targetSelector.add(4, new UniversalAngerGoal<>(this, false));
    }
 
