--- a/net/minecraft/entity/passive/PolarBearEntity.java
+++ b/net/minecraft/entity/passive/PolarBearEntity.java
@@ -84,8 +84,8 @@
       this.goalSelector.add(7, new LookAroundGoal(this));
       this.targetSelector.add(1, new PolarBearEntity.PolarBearRevengeGoal());
       this.targetSelector.add(2, new PolarBearEntity.ProtectBabiesGoal());
-      this.targetSelector.add(3, new ActiveTargetGoal(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
-      this.targetSelector.add(4, new ActiveTargetGoal(this, FoxEntity.class, 10, true, true, (Predicate<LivingEntity>)null));
+      this.targetSelector.add(3, new ActiveTargetGoal<>(this, PlayerEntity.class, 10, true, false, this::shouldAngerAt));
+      this.targetSelector.add(4, new ActiveTargetGoal<>(this, FoxEntity.class, 10, true, true, (Predicate<LivingEntity>)null));
       this.targetSelector.add(5, new UniversalAngerGoal<>(this, false));
    }
 
