--- a/net/minecraft/entity/passive/TadpoleEntity.java
+++ b/net/minecraft/entity/passive/TadpoleEntity.java
@@ -61,7 +61,7 @@
    }
 
    public Brain<TadpoleEntity> getBrain() {
-      return super.getBrain();
+      return (Brain<TadpoleEntity>) super.getBrain();
    }
 
    protected SoundEvent getFlopSound() {
@@ -193,8 +193,8 @@
    }
 
    private void growUp() {
-      World frogEntity = this.world;
-      if (frogEntity instanceof ServerWorld serverWorld) {
+      World world1 = this.world;
+      if (world1 instanceof ServerWorld serverWorld) {
          FrogEntity frogEntity = EntityType.FROG.create(this.world);
          if (frogEntity != null) {
             frogEntity.refreshPositionAndAngles(this.getX(), this.getY(), this.getZ(), this.getYaw(), this.getPitch());
