--- a/net/minecraft/screen/AbstractRecipeScreenHandler.java
+++ b/net/minecraft/screen/AbstractRecipeScreenHandler.java
@@ -13,7 +13,7 @@
    }
 
    public void fillInputSlots(boolean craftAll, Recipe<?> recipe, ServerPlayerEntity player) {
-      (new InputSlotFiller<>(this)).fillInputSlots(player, recipe, craftAll);
+      (new InputSlotFiller<>(this)).fillInputSlots(player, (Recipe<C>) recipe, craftAll);
    }
 
    public abstract void populateRecipeFinder(RecipeMatcher finder);
