--- a/net/minecraft/screen/LecternScreenHandler.java
+++ b/net/minecraft/screen/LecternScreenHandler.java
@@ -43,9 +43,11 @@
       } else {
          switch (id) {
             case 1:
+            {
                int i = this.propertyDelegate.get(0);
                this.setProperty(0, i - 1);
                return true;
+            }
             case 2:
                int i = this.propertyDelegate.get(0);
                this.setProperty(0, i + 1);
