--- a/net/minecraft/screen/SmithingScreenHandler.java
+++ b/net/minecraft/screen/SmithingScreenHandler.java
@@ -83,7 +83,7 @@
    }
 
    public int getSlotFor(ItemStack stack) {
-      return ((Optional)this.recipes.stream().map((recipe) -> getQuickMoveSlot(recipe, stack)).filter(Optional::isPresent).findFirst().orElse(Optional.of(0))).get();
+      return (int) ((Optional)this.recipes.stream().map((recipe) -> getQuickMoveSlot(recipe, stack)).filter(Optional::isPresent).findFirst().orElse(Optional.of(0))).get();
    }
 
    private static Optional<Integer> getQuickMoveSlot(SmithingRecipe recipe, ItemStack stack) {
