--- a/net/minecraft/data/dev/NbtProvider.java
+++ b/net/minecraft/data/dev/NbtProvider.java
@@ -64,7 +64,7 @@
                return var4;
             } catch (IOException var8) {
                LOGGER.error("Failed to read structure input directory", var8);
-               return CompletableFuture.completedFuture((Object)null);
+               return CompletableFuture.completedFuture(null);
             }
          }, Util.getMainWorkerExecutor()).thenCompose((future) -> future));
       }
