--- a/net/minecraft/entity/boss/dragon/EnderDragonFight.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/boss/dragon/EnderDragonFight.java	2023-03-30 21:03:20
@@ -143,7 +143,7 @@
       }
 
       NbtList nbtList = new NbtList();
-      ObjectListIterator var3 = this.gateways.iterator();
+      ObjectListIterator<Integer> var3 = this.gateways.iterator();
 
       while(var3.hasNext()) {
          int i = var3.next();
