--- a/java/net/minecraft/data/report/DynamicRegistriesProvider.java
+++ b/java/net/minecraft/data/report/DynamicRegistriesProvider.java
@@ -55,7 +55,7 @@
       Optional<JsonElement> optional = encoder.encodeStart(json, value).resultOrPartial((error) -> {
          LOGGER.error("Couldn't serialize element {}: {}", path, error);
       });
-      return optional.isPresent() ? DataProvider.writeToPath(cache, (JsonElement)optional.get(), path) : CompletableFuture.completedFuture((Object)null);
+      return optional.isPresent() ? DataProvider.writeToPath(cache, (JsonElement)optional.get(), path) : CompletableFuture.completedFuture(null);
    }
 
    public final String getName() {
