--- a/net/minecraft/client/resource/PeriodicNotificationManager.java
+++ b/net/minecraft/client/resource/PeriodicNotificationManager.java
@@ -124,10 +124,6 @@
 
    @Environment(EnvType.CLIENT)
    public static record Entry(long delay, long period, String title, String message) {
-      final long delay;
-      final long period;
-      final String title;
-      final String message;
 
       public Entry(long delay, long period, String title, String message) {
          this.delay = delay != 0L ? delay : period;
@@ -172,4 +168,4 @@
 
       }
    }
-}
+}
\ No newline at end of file
