--- a/net/minecraft/client/resource/ServerResourcePackProvider.java
+++ b/net/minecraft/client/resource/ServerResourcePackProvider.java
@@ -148,7 +148,7 @@
 
          this.downloadTask = null;
          if (this.serverContainer == null) {
-            return CompletableFuture.completedFuture((Object)null);
+            return CompletableFuture.completedFuture(null);
          }
 
          this.serverContainer = null;
@@ -215,6 +215,6 @@
 
    public CompletableFuture<Void> loadServerPack(LevelStorage.Session session) {
       Path path = session.getDirectory(WorldSavePath.RESOURCES_ZIP);
-      return Files.exists(path, new LinkOption[0]) && !Files.isDirectory(path, new LinkOption[0]) ? this.loadServerPack(path.toFile(), ResourcePackSource.WORLD) : CompletableFuture.completedFuture((Object)null);
+      return Files.exists(path, new LinkOption[0]) && !Files.isDirectory(path, new LinkOption[0]) ? this.loadServerPack(path.toFile(), ResourcePackSource.WORLD) : CompletableFuture.completedFuture(null);
    }
 }
