--- a/net/minecraft/util/math/intprovider/WeightedListIntProvider.java
+++ b/net/minecraft/util/math/intprovider/WeightedListIntProvider.java
@@ -8,7 +8,7 @@
 import net.minecraft.util.math.random.Random;
 
 public class WeightedListIntProvider extends IntProvider {
-   public static final Codec<WeightedListIntProvider> CODEC = RecordCodecBuilder.create((instance) -> instance.group(DataPool.createCodec(IntProvider.VALUE_CODEC).fieldOf("distribution").forGetter((provider) -> provider.weightedList)).apply(instance, WeightedListIntProvider::new));
+   public static final Codec<WeightedListIntProvider> CODEC = RecordCodecBuilder.create((instance) -> instance.group(DataPool.createDataCodec(IntProvider.VALUE_CODEC).fieldOf("distribution").forGetter((provider) -> provider.weightedList)).apply(instance, WeightedListIntProvider::new));
    private final DataPool<IntProvider> weightedList;
    private final int min;
    private final int max;
