--- a/java/net/minecraft/entity/projectile/ProjectileEntity.java
+++ b/java/net/minecraft/entity/projectile/ProjectileEntity.java
@@ -31,7 +31,7 @@
    private boolean leftOwner;
    private boolean shot;
 
-   ProjectileEntity(EntityType<? extends ProjectileEntity> entityType, World world) {
+   public ProjectileEntity(EntityType<? extends ProjectileEntity> entityType, World world) {
       super(entityType, world);
    }
 
@@ -170,8 +170,8 @@
       if (!entity.canBeHitByProjectile()) {
          return false;
       } else {
-         Entity entity = this.getOwner();
-         return entity == null || this.leftOwner || !entity.isConnectedThroughVehicle(entity);
+         Entity entity2 = this.getOwner();
+         return entity2 == null || this.leftOwner || !entity2.isConnectedThroughVehicle(entity);
       }
    }
 
@@ -182,7 +182,7 @@
       this.setYaw(updateRotation(this.prevYaw, (float)(MathHelper.atan2(vec3d.x, vec3d.z) * (double)(180F / (float)Math.PI))));
    }
 
-   protected static float updateRotation(float prevRot, float newRot) {
+   public static float updateRotation(float prevRot, float newRot) {
       while(newRot - prevRot < -180.0F) {
          prevRot -= 360.0F;
       }
