--- a/net/minecraft/test/StructureTestUtil.java
+++ b/net/minecraft/test/StructureTestUtil.java
@@ -199,7 +199,7 @@
       Comparator<BlockPos> comparator = Comparator.comparingInt((posx) -> posx.getManhattanDistance(pos));
       Collection<BlockPos> collection = findStructureBlocks(pos, radius, world);
       Optional<BlockPos> optional = collection.stream().min(comparator);
-      return (BlockPos)optional.orElse((Object)null);
+      return (BlockPos)optional.orElse(null);
    }
 
    public static Collection<BlockPos> findStructureBlocks(BlockPos pos, int radius, ServerWorld world) {
