--- a/net/minecraft/datafixer/fix/ChunkHeightAndBiomeFix.java
+++ b/net/minecraft/datafixer/fix/ChunkHeightAndBiomeFix.java
@@ -155,7 +155,7 @@
 
    private static Dynamic<?>[] fixBiomes(Dynamic<?> level, boolean overworld, int i, MutableBoolean heightAlreadyUpdated) {
       Dynamic<?>[] dynamics = new Dynamic[overworld ? 24 : 16];
-      int[] is = (int[])level.get("Biomes").asIntStreamOpt().result().map(IntStream::toArray).orElse((Object)null);
+      int[] is = (int[])level.get("Biomes").asIntStreamOpt().result().map(IntStream::toArray).orElse(null);
       if (is != null && is.length == 1536) {
          heightAlreadyUpdated.setValue(true);
 
