--- a/net/minecraft/datafixer/fix/StructuresToConfiguredStructuresFix.java
+++ b/net/minecraft/datafixer/fix/StructuresToConfiguredStructuresFix.java
@@ -111,7 +111,6 @@
    }
 
    static record Mapping(Map<String, String> biomeMapping, String fallback) {
-      final String fallback;
 
       public static StructuresToConfiguredStructuresFix.Mapping create(String mapping) {
          return new StructuresToConfiguredStructuresFix.Mapping(Map.of(), mapping);
@@ -131,4 +130,4 @@
          return builder.build();
       }
    }
-}
+}
\ No newline at end of file
