--- a/net/minecraft/entity/Entity.java	2023-03-30 19:29:27
+++ b/net/minecraft/entity/Entity.java	2023-03-30 19:29:44
@@ -2353,7 +2353,7 @@
                }
 
                return NetherPortal.getNetherTeleportTarget(destination, rect, axis, vec3d, this, this.getVelocity(), this.getYaw(), this.getPitch());
-            }).orElse((Object)null);
+            }).orElse(null);
          }
       } else {
          BlockPos blockPos;
@@ -2450,7 +2450,7 @@
 
    @Nullable
    public Text getCustomName() {
-      return (Text)((Optional)this.dataTracker.get(CUSTOM_NAME)).orElse((Object)null);
+      return (Text)((Optional)this.dataTracker.get(CUSTOM_NAME)).orElse(null);
    }
 
    public boolean hasCustomName() {
