--- a/net/minecraft/client/network/ClientPlayNetworkHandler.java
+++ b/net/minecraft/client/network/ClientPlayNetworkHandler.java
@@ -2419,6 +2419,5 @@
 
    @Environment(EnvType.CLIENT)
    static record QueuedPacket(Packet<ServerPlayPacketListener> packet, BooleanSupplier sendCondition, long expirationTime) {
-      final Packet<ServerPlayPacketListener> packet;
    }
-}
+}
\ No newline at end of file
