--- a/net/minecraft/world/timer/Timer.java
+++ b/net/minecraft/world/timer/Timer.java
@@ -29,7 +29,7 @@
    private final Table<String, Long, Timer.Event<T>> eventsByName = HashBasedTable.create();
 
    private static <T> Comparator<Timer.Event<T>> createEventComparator() {
-      return Comparator.comparingLong((event) -> event.triggerTime).thenComparing((event) -> event.id);
+      return Comparator.<Event<T>>comparingLong((event) -> event.triggerTime).thenComparing((event) -> event.id);
    }
 
    public Timer(TimerCallbackSerializer<T> timerCallbackSerializer, Stream<? extends Dynamic<?>> nbts) {
