--- a/java/net/minecraft/util/profiling/jfr/sample/NetworkIoStatistics.java
+++ b/java/net/minecraft/util/profiling/jfr/sample/NetworkIoStatistics.java
@@ -72,12 +72,10 @@
    }
 
    public static record PacketStatistics(long totalCount, long totalSize) {
-      final long totalCount;
-      final long totalSize;
       static final Comparator<NetworkIoStatistics.PacketStatistics> COMPARATOR = Comparator.comparing(NetworkIoStatistics.PacketStatistics::totalSize).thenComparing(NetworkIoStatistics.PacketStatistics::totalCount).reversed();
 
       NetworkIoStatistics.PacketStatistics add(NetworkIoStatistics.PacketStatistics statistics) {
          return new NetworkIoStatistics.PacketStatistics(this.totalCount + statistics.totalCount, this.totalSize + statistics.totalSize);
       }
    }
 }
