--- a/java/net/minecraft/util/profiling/jfr/sample/NetworkIoStatistics.java
+++ b/java/net/minecraft/util/profiling/jfr/sample/NetworkIoStatistics.java
@@ -72,8 +72,6 @@
    }
 
    public static record PacketStatistics(long totalCount, long totalSize) {
-      final long totalCount;
-      final long totalSize;
       static final Comparator<NetworkIoStatistics.PacketStatistics> COMPARATOR = Comparator.comparing(NetworkIoStatistics.PacketStatistics::totalSize).thenComparing(NetworkIoStatistics.PacketStatistics::totalCount).reversed();
 
       NetworkIoStatistics.PacketStatistics add(NetworkIoStatistics.PacketStatistics statistics) {
