--- a/net/minecraft/data/SnbtProvider.java
+++ b/net/minecraft/data/SnbtProvider.java
@@ -165,11 +165,6 @@
    }
 
    static record CompressedData(String name, byte[] bytes, @Nullable String snbtContent, HashCode sha1) {
-      final String name;
-      final byte[] bytes;
-      @Nullable
-      final String snbtContent;
-      final HashCode sha1;
    }
 
    static class CompressionException extends RuntimeException {
@@ -182,4 +177,4 @@
    public interface Tweaker {
       NbtCompound write(String name, NbtCompound nbt);
    }
-}
+}
\ No newline at end of file
