--- a/net/minecraft/server/network/ServerPlayNetworkHandler.java	2023-03-30 19:29:24
+++ b/net/minecraft/server/network/ServerPlayNetworkHandler.java	2023-03-30 19:29:44
@@ -1695,7 +1695,7 @@
       this.messageChainTaskQueue.append((executor) -> {
          this.player.setSession(session);
          this.server.getPlayerManager().sendToAll(new PlayerListS2CPacket(EnumSet.of(PlayerListS2CPacket.Action.INITIALIZE_CHAT), List.of(this.player)));
-         return CompletableFuture.completedFuture((Object)null);
+         return CompletableFuture.completedFuture(null);
       });
    }
 
