--- a/net/minecraft/server/network/ServerPlayerEntity.java	1970-01-12 08:58:04
+++ b/net/minecraft/server/network/ServerPlayerEntity.java	2023-03-30 19:40:07
@@ -303,7 +303,7 @@
    public void readCustomDataFromNbt(NbtCompound nbt) {
       super.readCustomDataFromNbt(nbt);
       if (nbt.contains("warden_spawn_tracker", 10)) {
-         SculkShriekerWarningManager.CODEC.parse(new Dynamic(NbtOps.INSTANCE, nbt.get("warden_spawn_tracker"))).resultOrPartial(LOGGER::error).ifPresent((sculkShriekerWarningManager) -> this.sculkShriekerWarningManager = sculkShriekerWarningManager);
+         SculkShriekerWarningManager.CODEC.parse(new Dynamic<>(NbtOps.INSTANCE, nbt.get("warden_spawn_tracker"))).resultOrPartial(LOGGER::error).ifPresent((sculkShriekerWarningManager) -> this.sculkShriekerWarningManager = sculkShriekerWarningManager);
       }
 
       if (nbt.contains("enteredNetherPosition", 10)) {
