--- a/net/minecraft/resource/fs/ResourceFileSystemProvider.java
+++ b/net/minecraft/resource/fs/ResourceFileSystemProvider.java
@@ -74,7 +74,7 @@
                   } catch (IOException var3) {
                      throw new DirectoryIteratorException(var3);
                   }
-               }).map((child) -> child).iterator();
+               }).map((child) -> (Path)child).iterator();
             }
 
             public void close() {
