--- a/net/minecraft/resource/fs/ResourcePath.java
+++ b/net/minecraft/resource/fs/ResourcePath.java
@@ -223,8 +223,8 @@
          ResourceFile resourcePath = this.file;
          if (resourcePath instanceof ResourceFile.Directory) {
             ResourceFile.Directory directory = (ResourceFile.Directory)resourcePath;
-            ResourcePath resourcePath = (ResourcePath)directory.children().get(name);
-            return resourcePath != null ? resourcePath : new ResourcePath(this.fileSystem, name, this, ResourceFile.EMPTY);
+            ResourcePath resourcePath2 = (ResourcePath)directory.children().get(name);
+            return resourcePath2 != null ? resourcePath2 : new ResourcePath(this.fileSystem, name, this, ResourceFile.EMPTY);
          } else if (this.file instanceof ResourceFile.File) {
             return new ResourcePath(this.fileSystem, name, this, ResourceFile.EMPTY);
          } else {
