--- a/net/minecraft/world/event/BlockPositionSource.java
+++ b/net/minecraft/world/event/BlockPositionSource.java
@@ -29,8 +29,8 @@
          return new BlockPositionSource(packetByteBuf.readBlockPos());
       }
 
-      public void writeToBuf(PacketByteBuf packetByteBuf, BlockPositionSource blockPositionSource) {
-         packetByteBuf.writeBlockPos(blockPositionSource.pos);
+      public <T extends PositionSource> void writeToBuf(PacketByteBuf packetByteBuf, T blockPositionSource) {
+         packetByteBuf.writeBlockPos(((BlockPositionSource)blockPositionSource).pos);
       }
 
       public Codec<BlockPositionSource> getCodec() {
