--- a/net/minecraft/world/storage/SerializingRegionBasedStorage.java	2023-03-30 19:29:27
+++ b/net/minecraft/world/storage/SerializingRegionBasedStorage.java	2023-03-30 19:29:45
@@ -116,7 +116,7 @@
    private void loadDataAt(ChunkPos pos) {
       Optional<NbtCompound> optional = (Optional)this.loadNbt(pos).join();
       RegistryOps<NbtElement> registryOps = RegistryOps.of(NbtOps.INSTANCE, this.dynamicRegistryManager);
-      this.update(pos, registryOps, (NbtElement)optional.orElse((Object)null));
+      this.update(pos, registryOps, (NbtElement)optional.orElse(null));
    }
 
    private CompletableFuture<Optional<NbtCompound>> loadNbt(ChunkPos pos) {
