--- a/net/minecraft/client/gui/screen/report/AbuseReportReasonScreen.java	2023-03-30 19:29:26
+++ b/net/minecraft/client/gui/screen/report/AbuseReportReasonScreen.java	2023-03-30 19:29:44
@@ -126,7 +126,7 @@
 
       @Nullable
       public AbuseReportReasonScreen.ReasonListWidget.ReasonEntry getEntry(AbuseReportReason reason) {
-         return (AbuseReportReasonScreen.ReasonListWidget.ReasonEntry)this.children().stream().filter((entry) -> entry.reason == reason).findFirst().orElse((Object)null);
+         return (AbuseReportReasonScreen.ReasonListWidget.ReasonEntry)this.children().stream().filter((entry) -> entry.reason == reason).findFirst().orElse(null);
       }
 
       public int getRowWidth() {
