--- a/net/minecraft/client/realms/FileUpload.java
+++ b/net/minecraft/client/realms/FileUpload.java
@@ -144,7 +144,7 @@
                JsonParser jsonParser = new JsonParser();
                JsonElement jsonElement = jsonParser.parse(string).getAsJsonObject().get("errorMsg");
                Optional<String> optional = Optional.ofNullable(jsonElement).map(JsonElement::getAsString);
-               uploadResultBuilder.withErrorMessage((String)optional.orElse((Object)null));
+               uploadResultBuilder.withErrorMessage((String)optional.orElse(null));
             } catch (Exception var8) {
             }
          }
