--- a/net/minecraft/entity/ai/WardenAngerManager.java	2023-03-30 19:29:27
+++ b/net/minecraft/entity/ai/WardenAngerManager.java	2023-03-30 19:29:44
@@ -155,7 +155,7 @@
 
    @Nullable
    private Entity getPrimeSuspectInternal() {
-      return (Entity)this.suspects.stream().filter(this.suspectPredicate).findFirst().orElse((Object)null);
+      return (Entity)this.suspects.stream().filter(this.suspectPredicate).findFirst().orElse(null);
    }
 
    public int getAngerFor(@Nullable Entity entity) {
