--- a/net/minecraft/client/render/WorldRenderer.java
+++ b/net/minecraft/client/render/WorldRenderer.java
@@ -483,7 +483,7 @@
          String string2 = "Failed to " + string + " shader: " + identifier;
          WorldRenderer.ProgramInitException programInitException = new WorldRenderer.ProgramInitException(string2, var8);
          if (this.client.getResourcePackManager().getEnabledNames().size() > 1) {
-            Text text = (Text)this.client.getResourceManager().streamResourcePacks().findFirst().map((resourcePack) -> Text.literal(resourcePack.getName())).orElse((Object)null);
+            Text text = (Text)this.client.getResourceManager().streamResourcePacks().findFirst().map((resourcePack) -> Text.literal(resourcePack.getName())).orElse(null);
             this.client.options.getGraphicsMode().setValue(GraphicsMode.FANCY);
             this.client.onResourceReloadFailure(programInitException, text);
          } else {
@@ -658,7 +658,7 @@
 
          this.chunkBuilder = null;
          this.noCullingBlockEntities.clear();
-         this.renderableChunks.set((Object)null);
+         this.renderableChunks.set(null);
          this.chunkInfos.clear();
       }
 
