--- a/java/net/minecraft/world/chunk/light/ChunkLightProvider.java
+++ b/java/net/minecraft/world/chunk/light/ChunkLightProvider.java
@@ -69,7 +69,7 @@
 
    private void clearChunkCache() {
       Arrays.fill(this.cachedChunkPositions, ChunkPos.MARKER);
-      Arrays.fill(this.cachedChunks, (Object)null);
+      Arrays.fill(this.cachedChunks, null);
    }
 
    protected BlockState getStateForLighting(long pos, @Nullable MutableInt opacity) {
