--- a/java/net/minecraft/world/chunk/light/LightingProvider.java
+++ b/java/net/minecraft/world/chunk/light/LightingProvider.java
@@ -52,9 +52,9 @@
 
    public int doLightUpdates(int i, boolean doSkylight, boolean skipEdgeLightPropagation) {
       if (this.blockLightProvider != null && this.skyLightProvider != null) {
-         int i = i / 2;
-         int j = this.blockLightProvider.doLightUpdates(i, doSkylight, skipEdgeLightPropagation);
-         int k = i - i + j;
+         int q = i / 2;
+         int j = this.blockLightProvider.doLightUpdates(q, doSkylight, skipEdgeLightPropagation);
+         int k = q - i + j;
          int l = this.skyLightProvider.doLightUpdates(k, doSkylight, skipEdgeLightPropagation);
          return j == 0 && l > 0 ? this.blockLightProvider.doLightUpdates(l, doSkylight, skipEdgeLightPropagation) : l;
       } else if (this.blockLightProvider != null) {
