--- a/net/minecraft/server/command/CommandManager.java
+++ b/net/minecraft/server/command/CommandManager.java
@@ -238,7 +238,7 @@
    private void makeTreeForSource(CommandNode<ServerCommandSource> tree, CommandNode<CommandSource> result, ServerCommandSource source, Map<CommandNode<ServerCommandSource>, CommandNode<CommandSource>> resultNodes) {
       for(CommandNode<ServerCommandSource> commandNode : tree.getChildren()) {
          if (commandNode.canUse(source)) {
-            ArgumentBuilder<CommandSource, ?> argumentBuilder = commandNode.createBuilder();
+            ArgumentBuilder<CommandSource, ?> argumentBuilder = result.createBuilder();
             argumentBuilder.requires((sourcex) -> true);
             if (argumentBuilder.getCommand() != null) {
                argumentBuilder.executes((context) -> 0);
