--- a/net/minecraft/server/command/SpawnArmorTrimsCommand.java	2023-03-30 19:29:25
+++ b/net/minecraft/server/command/SpawnArmorTrimsCommand.java	2023-03-30 19:29:45
@@ -70,7 +70,7 @@
       DefaultedList<ArmorTrim> defaultedList = DefaultedList.of();
       Registry<ArmorTrimPattern> registry = world.getRegistryManager().get(RegistryKeys.TRIM_PATTERN);
       Registry<ArmorTrimMaterial> registry2 = world.getRegistryManager().get(RegistryKeys.TRIM_MATERIAL);
-      registry.stream().sorted(Comparator.comparing((pattern) -> PATTERN_INDEX_GETTER.applyAsInt((RegistryKey)registry.getKey(pattern).orElse((Object)null)))).forEachOrdered((pattern) -> registry2.stream().sorted(Comparator.comparing((material) -> MATERIAL_INDEX_GETTER.applyAsInt((RegistryKey)registry2.getKey(material).orElse((Object)null)))).forEachOrdered((material) -> defaultedList.add(new ArmorTrim(registry2.getEntry(material), registry.getEntry(pattern)))));
+      registry.stream().sorted(Comparator.comparing((pattern) -> PATTERN_INDEX_GETTER.applyAsInt((RegistryKey)registry.getKey(pattern).orElse(null)))).forEachOrdered((pattern) -> registry2.stream().sorted(Comparator.comparing((material) -> MATERIAL_INDEX_GETTER.applyAsInt((RegistryKey)registry2.getKey(material).orElse(null)))).forEachOrdered((material) -> defaultedList.add(new ArmorTrim(registry2.getEntry(material), registry.getEntry(pattern)))));
       BlockPos blockPos = player.getBlockPos().offset(player.getHorizontalFacing(), 5);
       int i = ArmorMaterials.values().length - 1;
       double d = 3.0D;
