--- a/net/minecraft/client/render/model/ModelLoader.java	2023-03-30 19:29:26
+++ b/net/minecraft/client/render/model/ModelLoader.java	2023-03-30 19:29:45
@@ -199,7 +199,7 @@
 
    @Nullable
    static <T extends Comparable<T>> T getPropertyValue(Property<T> property, String string) {
-      return (T)(property.parse(string).orElse((Object)null));
+      return (T)(property.parse(string).orElse(null));
    }
 
    public UnbakedModel getOrLoadModel(Identifier id) {
@@ -476,7 +476,5 @@
 
    @Environment(EnvType.CLIENT)
    public static record SourceTrackedData(String source, JsonElement data) {
-      final String source;
-      final JsonElement data;
    }
-}
+}
\ No newline at end of file
