--- a/net/minecraft/client/render/model/SpriteAtlasManager.java
+++ b/net/minecraft/client/render/model/SpriteAtlasManager.java
@@ -44,8 +44,6 @@
 
    @Environment(EnvType.CLIENT)
    static record Atlas(SpriteAtlasTexture atlas, Identifier atlasInfoLocation) implements AutoCloseable {
-      final SpriteAtlasTexture atlas;
-      final Identifier atlasInfoLocation;
 
       public void close() {
          this.atlas.clear();
@@ -79,4 +77,4 @@
          this.atlasTexture.upload(this.stitchResult);
       }
    }
-}
+}
\ No newline at end of file
