--- a/net/minecraft/world/gen/OreVeinSampler.java
+++ b/net/minecraft/world/gen/OreVeinSampler.java
@@ -22,7 +22,7 @@
    private OreVeinSampler() {
    }
 
-   protected static ChunkNoiseSampler.BlockStateSampler create(DensityFunction veinToggle, DensityFunction veinRidged, DensityFunction veinGap, RandomSplitter randomDeriver) {
+   public static ChunkNoiseSampler.BlockStateSampler create(DensityFunction veinToggle, DensityFunction veinRidged, DensityFunction veinGap, RandomSplitter randomDeriver) {
       BlockState blockState = null;
       return (pos) -> {
          double d = veinToggle.sample(pos);
