--- a/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/ai/brain/sensor/GolemLastSeenSensor.java	2023-03-30 20:43:47
@@ -32,7 +32,7 @@
    public static void senseIronGolem(LivingEntity entity) {
       Optional<List<LivingEntity>> optional = entity.getBrain().getOptionalRegisteredMemory(MemoryModuleType.MOBS);
       if (optional.isPresent()) {
-         boolean bl = ((List)optional.get()).stream().anyMatch((livingEntity) -> livingEntity.getType().equals(EntityType.IRON_GOLEM));
+         boolean bl = (optional.get()).stream().anyMatch((livingEntity) -> livingEntity.getType().equals(EntityType.IRON_GOLEM));
          if (bl) {
             rememberIronGolem(entity);
          }
