--- a/net/minecraft/entity/ai/brain/sensor/PiglinBruteSpecificSensor.java
+++ b/net/minecraft/entity/ai/brain/sensor/PiglinBruteSpecificSensor.java
@@ -27,7 +27,7 @@
       LivingTargetCache livingTargetCache = (LivingTargetCache)brain.getOptionalRegisteredMemory(MemoryModuleType.VISIBLE_MOBS).orElse(LivingTargetCache.empty());
       Optional<MobEntity> optional = livingTargetCache.findFirst((livingEntityx) -> livingEntityx instanceof WitherSkeletonEntity || livingEntityx instanceof WitherEntity).map(MobEntity.class::cast);
 
-      for(LivingEntity livingEntity : (List)brain.getOptionalRegisteredMemory(MemoryModuleType.MOBS).orElse(ImmutableList.of())) {
+      for(LivingEntity livingEntity : brain.getOptionalRegisteredMemory(MemoryModuleType.MOBS).orElse(ImmutableList.of())) {
          if (livingEntity instanceof AbstractPiglinEntity && ((AbstractPiglinEntity)livingEntity).isAdult()) {
             list.add((AbstractPiglinEntity)livingEntity);
          }
