--- a/net/minecraft/world/ChunkSerializer.java	2023-03-30 19:29:26
+++ b/net/minecraft/world/ChunkSerializer.java	2023-03-30 19:29:44
@@ -140,7 +140,7 @@
       ChunkStatus.ChunkType chunkType = getChunkType(nbt);
       BlendingData blendingData;
       if (nbt.contains("blending_data", 10)) {
-         blendingData = (BlendingData)BlendingData.CODEC.parse(new Dynamic(NbtOps.INSTANCE, nbt.getCompound("blending_data"))).resultOrPartial(LOGGER::error).orElse((Object)null);
+         blendingData = (BlendingData)BlendingData.CODEC.parse(new Dynamic(NbtOps.INSTANCE, nbt.getCompound("blending_data"))).resultOrPartial(LOGGER::error).orElse(null);
       } else {
          blendingData = null;
       }
