--- a/net/minecraft/world/GameRules.java
+++ b/net/minecraft/world/GameRules.java
@@ -128,11 +128,11 @@
       return new GameRules((Map)this.rules.entrySet().stream().collect(ImmutableMap.toImmutableMap(Map.Entry::getKey, (entry) -> ((GameRules.Rule)entry.getValue()).copy())));
    }
 
-   public static void accept(GameRules.Visitor visitor) {
-      RULE_TYPES.forEach((key, type) -> accept(visitor, key, type));
+   public static <T extends GameRules.Rule<T>> void accept(GameRules.Visitor visitor) {
+      RULE_TYPES.forEach((key, type) -> accept(visitor, (GameRules.Key<T>)key, (GameRules.Type<T>)type));
    }
 
-   private static <T extends GameRules.Rule<T>> void accept(GameRules.Visitor consumer, GameRules.Key<?> key, GameRules.Type<?> type) {
+   private static <T extends GameRules.Rule<T>> void accept(GameRules.Visitor consumer, GameRules.Key<T> key, GameRules.Type<T> type) {
       consumer.visit(key, type);
       type.accept(consumer, key);
    }
