--- a/net/minecraft/world/GameMode.java
+++ b/net/minecraft/world/GameMode.java
@@ -16,7 +16,7 @@
 
    public static final GameMode DEFAULT = SURVIVAL;
    public static final StringIdentifiable.Codec<GameMode> CODEC = StringIdentifiable.createCodec(GameMode::values);
-   private static final IntFunction<GameMode> BY_ID = ValueLists.createIdToValueFunction(GameMode::getId, values(), ValueLists.OutOfBoundsHandling.ZERO);
+   private static final IntFunction<GameMode> BY_ID = ValueLists.<GameMode>createIdToValueFunction(g -> g.getId(), values(), ValueLists.OutOfBoundsHandling.ZERO);
    private static final int UNKNOWN = -1;
    private final int id;
    private final String name;
