--- a/net/minecraft/world/MobSpawnerLogic.java
+++ b/net/minecraft/world/MobSpawnerLogic.java
@@ -30,7 +30,7 @@
    private static final Logger LOGGER = LogUtils.getLogger();
    private static final int field_30951 = 1;
    private int spawnDelay = 20;
-   private DataPool<MobSpawnerEntry> spawnPotentials = DataPool.empty();
+   private DataPool<MobSpawnerEntry> spawnPotentials = DataPool.createEmpty();
    @Nullable
    private MobSpawnerEntry spawnEntry;
    private double field_9161;
@@ -186,7 +186,7 @@
       boolean bl2 = nbt.contains("SpawnPotentials", 9);
       if (bl2) {
          NbtList nbtList = nbt.getList("SpawnPotentials", 10);
-         this.spawnPotentials = (DataPool)MobSpawnerEntry.DATA_POOL_CODEC.parse(NbtOps.INSTANCE, nbtList).resultOrPartial((error) -> LOGGER.warn("Invalid SpawnPotentials list: {}", error)).orElseGet(DataPool::empty);
+         this.spawnPotentials = (DataPool)MobSpawnerEntry.DATA_POOL_CODEC.parse(NbtOps.INSTANCE, nbtList).resultOrPartial((error) -> LOGGER.warn("Invalid SpawnPotentials list: {}", error)).orElseGet(DataPool::createEmpty);
       } else {
          this.spawnPotentials = DataPool.of(this.spawnEntry != null ? this.spawnEntry : new MobSpawnerEntry());
       }
