--- a/java/net/minecraft/data/SnbtProvider.java
+++ b/java/net/minecraft/data/SnbtProvider.java
@@ -131,11 +131,6 @@
    }
 
    static record CompressedData(String name, byte[] bytes, @Nullable String snbtContent, HashCode sha1) {
-      final String name;
-      final byte[] bytes;
-      @Nullable
-      final String snbtContent;
-      final HashCode sha1;
    }
 
    static class CompressionException extends RuntimeException {
