--- a/net/minecraft/command/argument/SignedArgumentList.java
+++ b/net/minecraft/command/argument/SignedArgumentList.java
@@ -33,11 +33,11 @@
 
       for(ParsedCommandNode<S> parsedCommandNode : builder.getNodes()) {
          CommandNode parsedArgument = parsedCommandNode.getNode();
-         if (parsedArgument instanceof ArgumentCommandNode<S, ?> argumentCommandNode) {
+         if (parsedArgument instanceof ArgumentCommandNode argumentCommandNode) {
             if (argumentCommandNode.getType() instanceof SignedArgumentType) {
-               com.mojang.brigadier.context.ParsedArgument<S, ?> parsedArgument = (com.mojang.brigadier.context.ParsedArgument)builder.getArguments().get(argumentCommandNode.getName());
-               if (parsedArgument != null) {
-                  String string = parsedArgument.getRange().get(argumentName);
+               com.mojang.brigadier.context.ParsedArgument<S, ?> parsedArgument2 = (com.mojang.brigadier.context.ParsedArgument)builder.getArguments().get(argumentCommandNode.getName());
+               if (parsedArgument2 != null) {
+                  String string = parsedArgument2.getRange().get(argumentName);
                   list.add(new SignedArgumentList.ParsedArgument(argumentCommandNode, string));
                }
             }
