--- a/net/minecraft/block/sapling/SaplingGenerator.java
+++ b/net/minecraft/block/sapling/SaplingGenerator.java
@@ -22,7 +22,7 @@
       if (registryKey == null) {
          return false;
       } else {
-         RegistryEntry<ConfiguredFeature<?, ?>> registryEntry = (RegistryEntry)world.getRegistryManager().get(RegistryKeys.CONFIGURED_FEATURE).getEntry(registryKey).orElse((Object)null);
+         RegistryEntry<ConfiguredFeature<?, ?>> registryEntry = (RegistryEntry)world.getRegistryManager().get(RegistryKeys.CONFIGURED_FEATURE).getEntry(registryKey).orElse(null);
          if (registryEntry == null) {
             return false;
          } else {
