--- a/net/minecraft/entity/boss/WitherEntity.java
+++ b/net/minecraft/entity/boss/WitherEntity.java
@@ -475,7 +475,7 @@
    }
 
    public int getTrackedEntityId(int headIndex) {
-      return this.dataTracker.get((TrackedData)TRACKED_ENTITY_IDS.get(headIndex));
+      return (int) this.dataTracker.get((TrackedData)TRACKED_ENTITY_IDS.get(headIndex));
    }
 
    public void setTrackedEntityId(int headIndex, int id) {
