--- a/net/minecraft/client/texture/PlayerSkinProvider.java	2023-03-30 19:29:26
+++ b/net/minecraft/client/texture/PlayerSkinProvider.java	2023-03-30 19:29:45
@@ -134,7 +134,7 @@
    }
 
    public Map<MinecraftProfileTexture.Type, MinecraftProfileTexture> getTextures(GameProfile profile) {
-      Property property = (Property)Iterables.getFirst(profile.getProperties().get("textures"), (Object)null);
+      Property property = (Property)Iterables.getFirst(profile.getProperties().get("textures"), null);
       return (Map<MinecraftProfileTexture.Type, MinecraftProfileTexture>)(property == null ? ImmutableMap.of() : (Map)this.skinCache.getUnchecked(property.getValue()));
    }
 
