--- a/net/minecraft/client/texture/ResourceTexture.java
+++ b/net/minecraft/client/texture/ResourceTexture.java
@@ -104,7 +104,7 @@
             TextureResourceMetadata textureResourceMetadata = null;
 
             try {
-               textureResourceMetadata = (TextureResourceMetadata)resource.getMetadata().decode(TextureResourceMetadata.READER).orElse((Object)null);
+               textureResourceMetadata = (TextureResourceMetadata)resource.getMetadata().decode(TextureResourceMetadata.READER).orElse(null);
             } catch (RuntimeException var8) {
                ResourceTexture.LOGGER.warn("Failed reading metadata of: {}", id, var8);
             }
