--- a/net/minecraft/client/texture/TextureStitcher.java
+++ b/net/minecraft/client/texture/TextureStitcher.java
@@ -114,9 +114,6 @@
 
    @Environment(EnvType.CLIENT)
    static record Holder<T extends TextureStitcher.Stitchable>(T sprite, int width, int height) {
-      final T sprite;
-      final int width;
-      final int height;
 
       public Holder(T sprite, int mipLevel) {
          this(sprite, TextureStitcher.applyMipLevel(sprite.getWidth(), mipLevel), TextureStitcher.applyMipLevel(sprite.getHeight(), mipLevel));
@@ -225,4 +222,4 @@
 
       Identifier getId();
    }
-}
+}
\ No newline at end of file
