--- a/net/minecraft/client/texture/TextureManager.java
+++ b/net/minecraft/client/texture/TextureManager.java
@@ -143,7 +143,7 @@
          this.textures.put(id, asyncTexture);
          return asyncTexture.getLoadCompleteFuture().thenRunAsync(() -> this.registerTexture(id, asyncTexture), TextureManager::runOnRenderThread);
       } else {
-         return CompletableFuture.completedFuture((Object)null);
+         return CompletableFuture.completedFuture(null);
       }
    }
 
@@ -191,7 +191,7 @@
             }
          }
 
-         MinecraftClient.getInstance().send(() -> completableFuture.complete((Object)null));
+         MinecraftClient.getInstance().send(() -> completableFuture.complete(null));
       }, (runnable) -> RenderSystem.recordRenderCall(runnable::run));
       return completableFuture;
    }
