--- a/net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen.java	2023-03-30 19:29:27
+++ b/net/minecraft/client/gui/screen/CustomizeBuffetLevelScreen.java	2023-03-30 19:29:44
@@ -54,7 +54,7 @@
          this.client.setScreen(this.parent);
       }).dimensions(this.width / 2 - 155, this.height - 28, 150, 20).build());
       this.addDrawableChild(ButtonWidget.builder(ScreenTexts.CANCEL, (button) -> this.client.setScreen(this.parent)).dimensions(this.width / 2 + 5, this.height - 28, 150, 20).build());
-      this.biomeSelectionList.setSelected((CustomizeBuffetLevelScreen.BuffetBiomesListWidget.BuffetBiomeItem)this.biomeSelectionList.children().stream().filter((entry) -> Objects.equals(entry.biome, this.biome)).findFirst().orElse((Object)null));
+      this.biomeSelectionList.setSelected((CustomizeBuffetLevelScreen.BuffetBiomesListWidget.BuffetBiomeItem)this.biomeSelectionList.children().stream().filter((entry) -> Objects.equals(entry.biome, this.biome)).findFirst().orElse(null));
    }
 
    void refreshConfirmButton() {
