--- a/net/minecraft/entity/ai/brain/ScheduleBuilder.java
+++ b/net/minecraft/entity/ai/brain/ScheduleBuilder.java
@@ -19,7 +19,7 @@
    }
 
    public Schedule build() {
-      ((Set)this.activities.stream().map(ScheduleBuilder.ActivityEntry::getActivity).collect(Collectors.toSet())).forEach(this.schedule::addActivity);
+      (this.activities.stream().map(ScheduleBuilder.ActivityEntry::getActivity).collect(Collectors.toSet())).forEach(this.schedule::addActivity);
       this.activities.forEach((activity) -> {
          Activity activity2 = activity.getActivity();
          this.schedule.getOtherRules(activity2).forEach((rule) -> rule.add(activity.getStartTime(), 0.0F));
