--- a/net/minecraft/unused/packageinfo/PackageInfo6194.java
+++ b/net/minecraft/unused/packageinfo/PackageInfo6194.java
@@ -1,13 +0,0 @@
-package net.minecraft.unused.packageinfo;
-
-import javax.annotation.ParametersAreNonnullByDefault;
-import net.fabricmc.api.EnvType;
-import net.fabricmc.api.Environment;
-import net.minecraft.util.annotation.MethodsReturnNonnullByDefault;
-
-// $FF: synthetic class
-@ParametersAreNonnullByDefault
-@MethodsReturnNonnullByDefault
-@Environment(EnvType.CLIENT)
-interface PackageInfo6194 {
-}
