--- a/net/minecraft/unused/packageinfo/PackageInfo6077.java	1970-01-12 08:58:04
+++ b/net/minecraft/unused/packageinfo/PackageInfo6077.java	1970-01-01 10:00:00
@@ -1,12 +0,0 @@
-package net.minecraft.unused.packageinfo;
-
-import javax.annotation.ParametersAreNonnullByDefault;
-import net.minecraft.util.annotation.FieldsAreNonnullByDefault;
-import net.minecraft.util.annotation.MethodsReturnNonnullByDefault;
-
-// $FF: synthetic class
-@ParametersAreNonnullByDefault
-@MethodsReturnNonnullByDefault
-@FieldsAreNonnullByDefault
-interface PackageInfo6077 {
-}
