--- a/net/minecraft/unused/packageinfo/PackageInfo6060.java
+++ b/net/minecraft/unused/packageinfo/PackageInfo6060.java
@@ -1,12 +0,0 @@
-package net.minecraft.unused.packageinfo;
-
-import javax.annotation.ParametersAreNonnullByDefault;
-import net.minecraft.util.annotation.FieldsAreNonnullByDefault;
-import net.minecraft.util.annotation.MethodsReturnNonnullByDefault;
-
-// $FF: synthetic class
-@ParametersAreNonnullByDefault
-@MethodsReturnNonnullByDefault
-@FieldsAreNonnullByDefault
-interface PackageInfo6060 {
-}
