--- a/net/minecraft/util/function/ValueLists.java
+++ b/net/minecraft/util/function/ValueLists.java
@@ -38,7 +38,7 @@
          throw new IllegalArgumentException("Empty value list");
       } else {
          T[] objects = (T[])((Object[])values.clone());
-         Arrays.fill(objects, (Object)null);
+         Arrays.fill(objects, null);
 
          for(T object : values) {
             int j = valueToIndexFunction.applyAsInt(object);
