--- a/net/minecraft/util/collection/DefaultedList.java
+++ b/net/minecraft/util/collection/DefaultedList.java
@@ -25,7 +25,7 @@
       Validate.notNull(defaultValue);
       Object[] objects = new Object[size];
       Arrays.fill(objects, defaultValue);
-      return new DefaultedList<>(Arrays.asList(objects), defaultValue);
+      return (DefaultedList<E>) new DefaultedList<>(Arrays.asList(objects), defaultValue);
    }
 
    @SafeVarargs
