--- a/net/minecraft/client/util/DefaultSkinHelper.java
+++ b/net/minecraft/client/util/DefaultSkinHelper.java
@@ -39,11 +39,9 @@
 
    @Environment(EnvType.CLIENT)
    static record Skin(Identifier texture, DefaultSkinHelper.Model model) {
-      final Identifier texture;
-      final DefaultSkinHelper.Model model;
 
       public Skin(String texture, DefaultSkinHelper.Model model) {
          this(new Identifier(texture), model);
       }
    }
-}
+}
\ No newline at end of file
