--- a/net/minecraft/client/util/Untracker.java
+++ b/net/minecraft/client/util/Untracker.java
@@ -21,7 +21,7 @@
          method.setAccessible(true);
          Field field = Class.forName("org.lwjgl.system.MemoryUtil$LazyInit").getDeclaredField("ALLOCATOR");
          field.setAccessible(true);
-         Object object = field.get((Object)null);
+         Object object = field.get(null);
          return class_.isInstance(object) ? lookup.unreflect(method) : null;
       } catch (NoSuchMethodException | NoSuchFieldException | IllegalAccessException | ClassNotFoundException var5) {
          throw new RuntimeException(var5);
