--- a/net/minecraft/world/explosion/Explosion.java
+++ b/net/minecraft/world/explosion/Explosion.java
@@ -251,9 +251,9 @@
                BlockPos blockPos2 = blockPos.toImmutable();
                this.world.getProfiler().push("explosion_blocks");
                if (block.shouldDropItemsOnExplosion(this)) {
-                  World blockEntity = this.world;
-                  if (blockEntity instanceof ServerWorld) {
-                     ServerWorld serverWorld = (ServerWorld)blockEntity;
+                  World world1 = this.world;
+                  if (world1 instanceof ServerWorld) {
+                     ServerWorld serverWorld = (ServerWorld)world1;
                      BlockEntity blockEntity = blockState.hasBlockEntity() ? this.world.getBlockEntity(blockPos) : null;
                      LootContext.Builder builder = (new LootContext.Builder(serverWorld)).random(this.world.random).parameter(LootContextParameters.ORIGIN, Vec3d.ofCenter(blockPos)).parameter(LootContextParameters.TOOL, ItemStack.EMPTY).optionalParameter(LootContextParameters.BLOCK_ENTITY, blockEntity).optionalParameter(LootContextParameters.THIS_ENTITY, this.entity);
                      if (this.destructionType == Explosion.DestructionType.DESTROY_WITH_DECAY) {
