--- a/net/minecraft/util/annotation/ClientMethodsReturnNonnullByDefault.java
+++ b/net/minecraft/util/annotation/ClientMethodsReturnNonnullByDefault.java
@@ -3,13 +3,11 @@
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
-import javax.annotation.meta.TypeQualifierDefault;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import org.jetbrains.annotations.NotNull;
 
 @NotNull
-@TypeQualifierDefault({ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 @Environment(EnvType.CLIENT)
 public @interface ClientMethodsReturnNonnullByDefault {
