--- a/net/minecraft/block/ButtonBlock.java	2023-03-30 19:29:27
+++ b/net/minecraft/block/ButtonBlock.java	2023-03-30 19:29:44
@@ -159,7 +159,7 @@
    }
 
    protected void tryPowerWithProjectiles(BlockState state, World world, BlockPos pos) {
-      PersistentProjectileEntity persistentProjectileEntity = this.wooden ? (PersistentProjectileEntity)world.getNonSpectatingEntities(PersistentProjectileEntity.class, state.getOutlineShape(world, pos).getBoundingBox().offset(pos)).stream().findFirst().orElse((Object)null) : null;
+      PersistentProjectileEntity persistentProjectileEntity = this.wooden ? (PersistentProjectileEntity)world.getNonSpectatingEntities(PersistentProjectileEntity.class, state.getOutlineShape(world, pos).getBoundingBox().offset(pos)).stream().findFirst().orElse(null) : null;
       boolean bl = persistentProjectileEntity != null;
       boolean bl2 = state.get(POWERED);
       if (bl != bl2) {
