--- a/net/minecraft/block/PointedDripstoneBlock.java	2023-03-30 19:29:24
+++ b/net/minecraft/block/PointedDripstoneBlock.java	2023-03-30 19:29:45
@@ -389,7 +389,7 @@
       } else {
          Direction direction = state.get(VERTICAL_DIRECTION);
          BiPredicate<BlockPos, BlockState> biPredicate = (posx, statex) -> statex.isOf(Blocks.POINTED_DRIPSTONE) && statex.get(VERTICAL_DIRECTION) == direction;
-         return (BlockPos)searchInDirection(world, pos, direction.getDirection(), biPredicate, (statex) -> isTip(statex, allowMerged), range).orElse((Object)null);
+         return (BlockPos)searchInDirection(world, pos, direction.getDirection(), biPredicate, (statex) -> isTip(statex, allowMerged), range).orElse(null);
       }
    }
 
@@ -491,13 +491,13 @@
    private static BlockPos getCauldronPos(World world, BlockPos pos, Fluid fluid) {
       Predicate<BlockState> predicate = (state) -> state.getBlock() instanceof AbstractCauldronBlock && ((AbstractCauldronBlock)state.getBlock()).canBeFilledByDripstone(fluid);
       BiPredicate<BlockPos, BlockState> biPredicate = (posx, state) -> canDripThrough(world, posx, state);
-      return (BlockPos)searchInDirection(world, pos, Direction.DOWN.getDirection(), biPredicate, predicate, 11).orElse((Object)null);
+      return (BlockPos)searchInDirection(world, pos, Direction.DOWN.getDirection(), biPredicate, predicate, 11).orElse(null);
    }
 
    @Nullable
    public static BlockPos getDripPos(World world, BlockPos pos) {
       BiPredicate<BlockPos, BlockState> biPredicate = (posx, state) -> canDripThrough(world, posx, state);
-      return (BlockPos)searchInDirection(world, pos, Direction.UP.getDirection(), biPredicate, PointedDripstoneBlock::canDrip, 11).orElse((Object)null);
+      return (BlockPos)searchInDirection(world, pos, Direction.UP.getDirection(), biPredicate, PointedDripstoneBlock::canDrip, 11).orElse(null);
    }
 
    public static Fluid getDripFluid(ServerWorld world, BlockPos pos) {
@@ -568,8 +568,5 @@
    }
 
    static record DrippingFluid(BlockPos pos, Fluid fluid, BlockState sourceState) {
-      final BlockPos pos;
-      final Fluid fluid;
-      final BlockState sourceState;
    }
-}
+}
\ No newline at end of file
