--- a/net/minecraft/world/gen/chunk/GenerationShapeConfig.java	1970-01-12 08:58:04
+++ b/net/minecraft/world/gen/chunk/GenerationShapeConfig.java	2023-03-30 21:03:20
@@ -9,7 +9,7 @@
 import net.minecraft.world.dimension.DimensionType;
 
 public record GenerationShapeConfig(int minimumY, int height, int horizontalSize, int verticalSize) {
-   public static final Codec<GenerationShapeConfig> CODEC = RecordCodecBuilder.create((instance) -> instance.group(Codec.intRange(DimensionType.MIN_HEIGHT, DimensionType.MAX_COLUMN_HEIGHT).fieldOf("min_y").forGetter(GenerationShapeConfig::minimumY), Codec.intRange(0, DimensionType.MAX_HEIGHT).fieldOf("height").forGetter(GenerationShapeConfig::height), Codec.intRange(1, 4).fieldOf("size_horizontal").forGetter(GenerationShapeConfig::horizontalSize), Codec.intRange(1, 4).fieldOf("size_vertical").forGetter(GenerationShapeConfig::verticalSize)).apply(instance, GenerationShapeConfig::new)).comapFlatMap(GenerationShapeConfig::checkHeight, Function.identity());
+   public static final Codec<GenerationShapeConfig> CODEC = RecordCodecBuilder.<GenerationShapeConfig>create((instance) -> instance.group(Codec.intRange(DimensionType.MIN_HEIGHT, DimensionType.MAX_COLUMN_HEIGHT).fieldOf("min_y").forGetter(GenerationShapeConfig::minimumY), Codec.intRange(0, DimensionType.MAX_HEIGHT).fieldOf("height").forGetter(GenerationShapeConfig::height), Codec.intRange(1, 4).fieldOf("size_horizontal").forGetter(GenerationShapeConfig::horizontalSize), Codec.intRange(1, 4).fieldOf("size_vertical").forGetter(GenerationShapeConfig::verticalSize)).apply(instance, GenerationShapeConfig::new)).comapFlatMap(GenerationShapeConfig::checkHeight, Function.identity());
    protected static final GenerationShapeConfig SURFACE = create(-64, 384, 1, 2);
    protected static final GenerationShapeConfig NETHER = create(0, 128, 1, 2);
    protected static final GenerationShapeConfig END = create(0, 128, 2, 1);
