--- a/net/minecraft/client/font/BitmapFont.java	1970-01-12 08:58:04
+++ b/net/minecraft/client/font/BitmapFont.java	2023-03-30 19:24:17
@@ -48,13 +48,6 @@
 
    @Environment(EnvType.CLIENT)
    static record BitmapFontGlyph(float scaleFactor, NativeImage image, int x, int y, int width, int height, int advance, int ascent) implements Glyph {
-      final float scaleFactor;
-      final NativeImage image;
-      final int x;
-      final int y;
-      final int width;
-      final int height;
-      final int ascent;
 
       public float getAdvance() {
          return (float)this.advance;
@@ -202,4 +195,4 @@
          return i + 1;
       }
    }
-}
+}
\ No newline at end of file
