--- a/net/minecraft/client/font/UnicodeTextureFont.java
+++ b/net/minecraft/client/font/UnicodeTextureFont.java
@@ -251,11 +251,6 @@
 
    @Environment(EnvType.CLIENT)
    static record UnicodeTextureGlyph(int unpackSkipPixels, int unpackSkipRows, int width, int height, NativeImage image) implements Glyph {
-      final int unpackSkipPixels;
-      final int unpackSkipRows;
-      final int width;
-      final int height;
-      final NativeImage image;
 
       public float getAdvance() {
          return (float)(this.width / 2 + 1);
@@ -293,4 +288,4 @@
          });
       }
    }
-}
+}
\ No newline at end of file
