--- a/net/minecraft/registry/Registries.java
+++ b/net/minecraft/registry/Registries.java
@@ -194,8 +194,8 @@
 
    private static <T, R extends MutableRegistry<T>> R create(RegistryKey<? extends Registry<T>> key, R registry, Registries.Initializer<T> initializer, Lifecycle lifecycle) {
       Identifier identifier = key.getValue();
-      DEFAULT_ENTRIES.put(identifier, (Supplier)() -> initializer.run(registry));
-      ROOT.add(key, registry, lifecycle);
+      DEFAULT_ENTRIES.put(identifier, () -> initializer.run(registry));
+      ROOT.add((RegistryKey<MutableRegistry<?>>) key, registry, lifecycle);
       return registry;
    }
 
