--- a/net/minecraft/registry/RegistryLoader.java
+++ b/net/minecraft/registry/RegistryLoader.java
@@ -97,7 +97,7 @@
       Map<Identifier, Map<Identifier, Exception>> map = (Map)exceptions.entrySet().stream().collect(Collectors.groupingBy((entry) -> ((RegistryKey)entry.getKey()).getRegistry(), Collectors.toMap((entry) -> ((RegistryKey)entry.getKey()).getValue(), Map.Entry::getValue)));
       map.entrySet().stream().sorted(java.util.Map.Entry.comparingByKey()).forEach((entry) -> {
          printWriter.printf("> Errors in registry %s:%n", entry.getKey());
-         ((Map)entry.getValue()).entrySet().stream().sorted(java.util.Map.Entry.comparingByKey()).forEach((elementEntry) -> {
+         (entry.getValue()).entrySet().stream().sorted(java.util.Map.Entry.comparingByKey()).forEach((elementEntry) -> {
             printWriter.printf(">> Errors in element %s:%n", elementEntry.getKey());
             ((Exception)elementEntry.getValue()).printStackTrace(printWriter);
          });
