--- a/net/minecraft/registry/RegistryKey.java
+++ b/net/minecraft/registry/RegistryKey.java
@@ -41,7 +41,7 @@
    }
 
    public <E> Optional<RegistryKey<E>> tryCast(RegistryKey<? extends Registry<E>> registryRef) {
-      return this.isOf(registryRef) ? Optional.of(this) : Optional.empty();
+      return this.isOf(registryRef) ? Optional.of((RegistryKey<E>) this) : Optional.empty();
    }
 
    public Identifier getValue() {
@@ -53,7 +53,5 @@
    }
 
    static record RegistryIdPair(Identifier registry, Identifier id) {
-      final Identifier registry;
-      final Identifier id;
    }
-}
+}
\ No newline at end of file
