--- a/net/minecraft/world/updater/WorldUpdater.java
+++ b/net/minecraft/world/updater/WorldUpdater.java
@@ -125,7 +125,7 @@
                   boolean bl2 = false;
 
                   try {
-                     NbtCompound nbtCompound = (NbtCompound)((Optional)versionedChunkStorage.getNbt(chunkPos).join()).orElse((Object)null);
+                     NbtCompound nbtCompound = (NbtCompound)((Optional)versionedChunkStorage.getNbt(chunkPos).join()).orElse(null);
                      if (nbtCompound != null) {
                         int i = VersionedChunkStorage.getDataVersion(nbtCompound);
                         ChunkGenerator chunkGenerator = ((DimensionOptions)this.dimensionOptionsRegistry.getOrThrow(RegistryKeys.toDimensionKey(registryKey3))).chunkGenerator();
