--- a/net/minecraft/util/Util.java	2023-03-30 19:29:26
+++ b/net/minecraft/util/Util.java	2023-03-30 19:29:45
@@ -385,7 +385,7 @@
       CompletableFuture<?>[] completableFutures = new CompletableFuture[futures.size()];
       futures.forEach((future) -> {
          int i = list.size();
-         list.add((Object)null);
+         list.add(null);
          completableFutures[i] = future.whenComplete((value, throwable) -> {
             if (throwable != null) {
                exceptionHandler.accept(throwable);
