--- a/net/minecraft/item/ItemGroup.java
+++ b/net/minecraft/item/ItemGroup.java
@@ -196,7 +196,6 @@
    }
 
    public static record DisplayContext(FeatureSet enabledFeatures, boolean hasPermissions, RegistryWrapper.WrapperLookup lookup) {
-      final FeatureSet enabledFeatures;
 
       public boolean doesNotMatch(FeatureSet enabledFeatures, boolean hasPermissions, RegistryWrapper.WrapperLookup lookup) {
          return !this.enabledFeatures.equals(enabledFeatures) || this.hasPermissions != hasPermissions || this.lookup != lookup;
@@ -287,4 +286,4 @@
       HOTBAR,
       SEARCH;
    }
-}
+}
\ No newline at end of file
