--- a/java/net/minecraft/unused/packageinfo/PackageInfo6078.java
+++ b/java/net/minecraft/unused/packageinfo/PackageInfo6078.java
@@ -1,12 +0,0 @@
-package net.minecraft.unused.packageinfo;
-
-import javax.annotation.ParametersAreNonnullByDefault;
-import net.minecraft.util.annotation.FieldsAreNonnullByDefault;
-import net.minecraft.util.annotation.MethodsReturnNonnullByDefault;
-
-// $FF: synthetic class
-@ParametersAreNonnullByDefault
-@FieldsAreNonnullByDefault
-@MethodsReturnNonnullByDefault
-interface PackageInfo6078 {
-}
