--- a/java/net/minecraft/world/level/storage/SessionLock.java
+++ b/java/net/minecraft/world/level/storage/SessionLock.java
@@ -18,16 +18,16 @@
    private static final ByteBuffer SNOWMAN;
 
    public static SessionLock create(Path path) throws IOException {
-      Path path = path.resolve("session.lock");
+      Path path2 = path.resolve("session.lock");
       PathUtil.createDirectories(path);
-      FileChannel filechannel = FileChannel.open(path, StandardOpenOption.CREATE, StandardOpenOption.WRITE);
+      FileChannel filechannel = FileChannel.open(path2, StandardOpenOption.CREATE, StandardOpenOption.WRITE);
 
       try {
          filechannel.write(SNOWMAN.duplicate());
          filechannel.force(true);
          FileLock filelock = filechannel.tryLock();
          if (filelock == null) {
-            throw SessionLock.AlreadyLockedException.create(path);
+            throw SessionLock.AlreadyLockedException.create(path2);
          } else {
             return new SessionLock(filechannel, filelock);
          }
@@ -66,12 +66,12 @@
    }
 
    public static boolean isLocked(Path path) throws IOException {
-      Path path = path.resolve("session.lock");
+      Path path2 = path.resolve("session.lock");
 
       try {
          boolean flag;
          try (
-            FileChannel filechannel = FileChannel.open(path, StandardOpenOption.WRITE);
+            FileChannel filechannel = FileChannel.open(path2, StandardOpenOption.WRITE);
             FileLock filelock = filechannel.tryLock();
          ) {
             flag = filelock == null;
