--- a/net/minecraft/block/entity/AbstractFurnaceBlockEntity.java
+++ b/net/minecraft/block/entity/AbstractFurnaceBlockEntity.java
@@ -242,7 +242,7 @@
       if (blockEntity.isBurning() || bl4 && bl3) {
          Recipe<?> recipe;
          if (bl3) {
-            recipe = (Recipe)blockEntity.matchGetter.getFirstMatch(blockEntity, world).orElse((Object)null);
+            recipe = (Recipe)blockEntity.matchGetter.getFirstMatch(blockEntity, world).orElse(null);
          } else {
             recipe = null;
          }
