--- a/net/minecraft/block/entity/BannerPattern.java
+++ b/net/minecraft/block/entity/BannerPattern.java
@@ -30,7 +30,7 @@
 
    @Nullable
    public static RegistryEntry<BannerPattern> byId(String id) {
-      return (RegistryEntry)Registries.BANNER_PATTERN.streamEntries().filter((pattern) -> ((BannerPattern)pattern.value()).id.equals(id)).findAny().orElse((Object)null);
+      return (RegistryEntry)Registries.BANNER_PATTERN.streamEntries().filter((pattern) -> ((BannerPattern)pattern.value()).id.equals(id)).findAny().orElse(null);
    }
 
    public static class Patterns {
