--- a/net/minecraft/block/entity/SculkShriekerBlockEntity.java
+++ b/net/minecraft/block/entity/SculkShriekerBlockEntity.java
@@ -71,7 +71,7 @@
       }
 
       if (nbt.contains("listener", 10)) {
-         VibrationListener.createCodec(this).parse(new Dynamic(NbtOps.INSTANCE, nbt.getCompound("listener"))).resultOrPartial(LOGGER::error).ifPresent((vibrationListener) -> this.vibrationListener = vibrationListener);
+         VibrationListener.createCodec(this).parse(new Dynamic(NbtOps.INSTANCE, nbt.getCompound("listener"))).resultOrPartial(a -> LOGGER.error((String) a)).ifPresent((vibrationListener) -> this.vibrationListener = (VibrationListener) vibrationListener);
       }
 
    }
