--- a/net/minecraft/block/entity/SculkCatalystBlockEntity.java
+++ b/net/minecraft/block/entity/SculkCatalystBlockEntity.java
@@ -45,9 +45,9 @@
          if (i instanceof LivingEntity) {
             LivingEntity livingEntity = (LivingEntity)i;
             if (!livingEntity.isExperienceDroppingDisabled()) {
-               int i = livingEntity.getXpToDrop();
-               if (livingEntity.shouldDropXp() && i > 0) {
-                  this.spreadManager.spread(BlockPos.ofFloored(emitterPos.offset(Direction.UP, 0.5D)), i);
+               int i2 = livingEntity.getXpToDrop();
+               if (livingEntity.shouldDropXp() && i2 > 0) {
+                  this.spreadManager.spread(BlockPos.ofFloored(emitterPos.offset(Direction.UP, 0.5D)), i2);
                   this.triggerCriteria(livingEntity);
                }
 
