--- a/net/minecraft/block/entity/SkullBlockEntity.java	2023-03-30 19:29:28
+++ b/net/minecraft/block/entity/SkullBlockEntity.java	2023-03-30 19:29:45
@@ -133,7 +133,7 @@
    public static void loadProperties(@Nullable GameProfile owner, Consumer<GameProfile> callback) {
       if (owner != null && !StringHelper.isEmpty(owner.getName()) && (!owner.isComplete() || !owner.getProperties().containsKey("textures")) && userCache != null && sessionService != null) {
          userCache.findByNameAsync(owner.getName(), (profile) -> Util.getMainWorkerExecutor().execute(() -> Util.ifPresentOrElse(profile, (profilex) -> {
-                  Property property = (Property)Iterables.getFirst(profilex.getProperties().get("textures"), (Object)null);
+                  Property property = (Property)Iterables.getFirst(profilex.getProperties().get("textures"), null);
                   if (property == null) {
                      MinecraftSessionService minecraftSessionService = sessionService;
                      if (minecraftSessionService == null) {
