--- a/net/minecraft/block/entity/StructureBlockBlockEntity.java
+++ b/net/minecraft/block/entity/StructureBlockBlockEntity.java
@@ -260,7 +260,7 @@
          return false;
       } else {
          BlockPos blockPos = this.getPos();
-         int i = 80;
+         // int i = 80;
          BlockPos blockPos2 = new BlockPos(blockPos.getX() - 80, this.world.getBottomY(), blockPos.getZ() - 80);
          BlockPos blockPos3 = new BlockPos(blockPos.getX() + 80, this.world.getTopY() - 1, blockPos.getZ() + 80);
          Stream<BlockPos> stream = this.streamCornerPos(blockPos2, blockPos3);
