--- a/net/minecraft/registry/entry/RegistryEntryList.java
+++ b/net/minecraft/registry/entry/RegistryEntryList.java
@@ -126,12 +126,12 @@
       private final TagKey<T> tag;
       private List<RegistryEntry<T>> entries = List.of();
 
-      Named(RegistryEntryOwner<T> owner, TagKey<T> tag) {
+      public Named(RegistryEntryOwner<T> owner, TagKey<T> tag) {
          this.owner = owner;
          this.tag = tag;
       }
 
-      void copyOf(List<RegistryEntry<T>> entries) {
+      public void copyOf(List<RegistryEntry<T>> entries) {
          this.entries = List.copyOf(entries);
       }
 
