--- a/net/minecraft/registry/entry/RegistryEntry.java
+++ b/net/minecraft/registry/entry/RegistryEntry.java
@@ -162,7 +162,7 @@
          return this.registryKey != null && this.value != null;
       }
 
-      void setRegistryKey(RegistryKey<T> registryKey) {
+      public void setRegistryKey(RegistryKey<T> registryKey) {
          if (this.registryKey != null && registryKey != this.registryKey) {
             throw new IllegalStateException("Can't change holder key: existing=" + this.registryKey + ", new=" + registryKey);
          } else {
@@ -170,7 +170,7 @@
          }
       }
 
-      void setValue(T value) {
+      public void setValue(T value) {
          if (this.referenceType == RegistryEntry.Reference.Type.INTRUSIVE && this.value != value) {
             throw new IllegalStateException("Can't change holder " + this.registryKey + " value: existing=" + this.value + ", new=" + value);
          } else {
@@ -178,7 +178,7 @@
          }
       }
 
-      void setTags(Collection<TagKey<T>> tags) {
+      public void setTags(Collection<TagKey<T>> tags) {
          this.tags = Set.copyOf(tags);
       }
 
