--- a/net/minecraft/loot/entry/GroupEntry.java
+++ b/net/minecraft/loot/entry/GroupEntry.java
@@ -29,8 +29,8 @@
             };
          default:
             return (context, lootChoiceExpander) -> {
-               for(EntryCombiner entryCombiner : children) {
-                  entryCombiner.expand(context, lootChoiceExpander);
+               for(EntryCombiner entryCombiner3 : children) {
+                  entryCombiner3.expand(context, lootChoiceExpander);
                }
 
                return true;
