--- a/java/net/minecraft/resource/fs/ResourceFileSystem.java
+++ b/java/net/minecraft/resource/fs/ResourceFileSystem.java
@@ -169,11 +169,9 @@
    }
 
    static record Directory(Map<String, ResourceFileSystem.Directory> children, Map<String, Path> files) {
-      final Map<String, ResourceFileSystem.Directory> children;
-      final Map<String, Path> files;
 
       public Directory() {
          this(new HashMap<>(), new HashMap<>());
       }
    }
 }
