--- a/net/minecraft/command/EntitySelectorOptions.java
+++ b/net/minecraft/command/EntitySelectorOptions.java
@@ -486,8 +486,5 @@
    }
 
    static record SelectorOption(EntitySelectorOptions.SelectorHandler handler, Predicate<EntitySelectorReader> condition, Text description) {
-      final EntitySelectorOptions.SelectorHandler handler;
-      final Predicate<EntitySelectorReader> condition;
-      final Text description;
    }
-}
+}
\ No newline at end of file
