--- a/net/minecraft/recipe/CuttingRecipe.java
+++ b/net/minecraft/recipe/CuttingRecipe.java
@@ -90,7 +90,8 @@
          return this.recipeFactory.create(identifier, string, ingredient, itemStack);
       }
 
-      public void write(PacketByteBuf packetByteBuf, T cuttingRecipe) {
+      public <T extends Recipe> void write(PacketByteBuf packetByteBuf, T recipe) {
+         CuttingRecipe cuttingRecipe = (CuttingRecipe) recipe;
          packetByteBuf.writeString(cuttingRecipe.group);
          cuttingRecipe.input.write(packetByteBuf);
          packetByteBuf.writeItemStack(cuttingRecipe.output);
