--- a/net/minecraft/recipe/LegacySmithingRecipe.java
+++ b/net/minecraft/recipe/LegacySmithingRecipe.java
@@ -89,7 +89,8 @@
          return new LegacySmithingRecipe(identifier, ingredient, ingredient2, itemStack);
       }
 
-      public void write(PacketByteBuf packetByteBuf, LegacySmithingRecipe legacySmithingRecipe) {
+      public void write(PacketByteBuf packetByteBuf, Recipe recipe) {
+         LegacySmithingRecipe legacySmithingRecipe = (LegacySmithingRecipe)recipe;
          legacySmithingRecipe.base.write(packetByteBuf);
          legacySmithingRecipe.addition.write(packetByteBuf);
          packetByteBuf.writeItemStack(legacySmithingRecipe.result);
