--- a/net/minecraft/recipe/RecipeSerializer.java
+++ b/net/minecraft/recipe/RecipeSerializer.java
@@ -27,6 +27,7 @@
    RecipeSerializer<SmokingRecipe> SMOKING = register("smoking", new CookingRecipeSerializer<>(SmokingRecipe::new, 100));
    RecipeSerializer<CampfireCookingRecipe> CAMPFIRE_COOKING = register("campfire_cooking", new CookingRecipeSerializer<>(CampfireCookingRecipe::new, 100));
    RecipeSerializer<StonecuttingRecipe> STONECUTTING = register("stonecutting", new CuttingRecipe.Serializer<>(StonecuttingRecipe::new));
+   @SuppressWarnings("removal")
    RecipeSerializer<LegacySmithingRecipe> SMITHING = register("smithing", new LegacySmithingRecipe.Serializer());
    RecipeSerializer<SmithingTransformRecipe> SMITHING_TRANSFORM = register("smithing_transform", new SmithingTransformRecipe.Serializer());
    RecipeSerializer<SmithingTrimRecipe> SMITHING_TRIM = register("smithing_trim", new SmithingTrimRecipe.Serializer());
