--- a/net/minecraft/recipe/ShapedRecipe.java
+++ b/net/minecraft/recipe/ShapedRecipe.java
@@ -310,7 +310,8 @@
          return new ShapedRecipe(identifier, string, craftingRecipeCategory, i, j, defaultedList, itemStack, bl);
       }
 
-      public void write(PacketByteBuf packetByteBuf, ShapedRecipe shapedRecipe) {
+      public void write(PacketByteBuf packetByteBuf, ShapedRecipe recipe) {
+         ShapedRecipe shapedRecipe = (ShapedRecipe)recipe;
          packetByteBuf.writeVarInt(shapedRecipe.width);
          packetByteBuf.writeVarInt(shapedRecipe.height);
          packetByteBuf.writeString(shapedRecipe.group);
