--- a/net/minecraft/recipe/SmithingTransformRecipe.java
+++ b/net/minecraft/recipe/SmithingTransformRecipe.java
@@ -85,7 +85,8 @@
          return new SmithingTransformRecipe(identifier, ingredient, ingredient2, ingredient3, itemStack);
       }
 
-      public void write(PacketByteBuf packetByteBuf, SmithingTransformRecipe smithingTransformRecipe) {
+      public void write(PacketByteBuf packetByteBuf, Recipe recipe) {
+         SmithingTransformRecipe smithingTransformRecipe = (SmithingTransformRecipe) recipe;
          smithingTransformRecipe.template.write(packetByteBuf);
          smithingTransformRecipe.base.write(packetByteBuf);
          smithingTransformRecipe.addition.write(packetByteBuf);
