--- a/net/minecraft/recipe/SmithingTrimRecipe.java
+++ b/net/minecraft/recipe/SmithingTrimRecipe.java
@@ -112,7 +112,8 @@
          return new SmithingTrimRecipe(identifier, ingredient, ingredient2, ingredient3);
       }
 
-      public void write(PacketByteBuf packetByteBuf, SmithingTrimRecipe smithingTrimRecipe) {
+      public void write(PacketByteBuf packetByteBuf, Recipe recipe) {
+         SmithingTrimRecipe smithingTrimRecipe = (SmithingTrimRecipe) recipe;
          smithingTrimRecipe.template.write(packetByteBuf);
          smithingTrimRecipe.base.write(packetByteBuf);
          smithingTrimRecipe.addition.write(packetByteBuf);
