--- a/java/net/minecraft/village/VillagerGossips.java
+++ b/java/net/minecraft/village/VillagerGossips.java
@@ -181,9 +181,6 @@
    }
 
    static record GossipEntry(UUID target, VillageGossipType type, int value) {
-      final UUID target;
-      final VillageGossipType type;
-      final int value;
       public static final Codec<VillagerGossips.GossipEntry> CODEC = RecordCodecBuilder.create((instance) -> {
          return instance.group(Uuids.INT_STREAM_CODEC.fieldOf("Target").forGetter(VillagerGossips.GossipEntry::target), VillageGossipType.CODEC.fieldOf("Type").forGetter(VillagerGossips.GossipEntry::type), Codecs.POSITIVE_INT.fieldOf("Value").forGetter(VillagerGossips.GossipEntry::value)).apply(instance, VillagerGossips.GossipEntry::new);
       });
@@ -246,4 +243,4 @@
          this.associatedGossip.removeInt(gossipType);
       }
    }
 }
