--- a/java/net/minecraft/client/texture/atlas/UnstitchAtlasSource.java
+++ b/java/net/minecraft/client/texture/atlas/UnstitchAtlasSource.java
@@ -62,11 +62,6 @@
    }
 
    static record Region(Identifier sprite, double x, double y, double width, double height) {
-      final Identifier sprite;
-      final double x;
-      final double y;
-      final double width;
-      final double height;
       public static final Codec<UnstitchAtlasSource.Region> CODEC = RecordCodecBuilder.create((instance) -> {
          return instance.group(Identifier.CODEC.fieldOf("sprite").forGetter(UnstitchAtlasSource.Region::sprite), Codec.DOUBLE.fieldOf("x").forGetter(UnstitchAtlasSource.Region::x), Codec.DOUBLE.fieldOf("y").forGetter(UnstitchAtlasSource.Region::y), Codec.DOUBLE.fieldOf("width").forGetter(UnstitchAtlasSource.Region::width), Codec.DOUBLE.fieldOf("height").forGetter(UnstitchAtlasSource.Region::height)).apply(instance, UnstitchAtlasSource.Region::new);
       });
