--- a/net/minecraft/entity/vehicle/AbstractMinecartEntity.java
+++ b/net/minecraft/entity/vehicle/AbstractMinecartEntity.java
@@ -160,7 +160,7 @@
             EntityPose entityPose = (EntityPose)e.next();
             EntityDimensions entityDimensions = passenger.getDimensions(entityPose);
             float f = Math.min(entityDimensions.width, 1.0F) / 2.0F;
-            UnmodifiableIterator g = ((ImmutableList)DISMOUNT_FREE_Y_SPACES_NEEDED.get(entityPose)).iterator();
+            UnmodifiableIterator<Integer> g = DISMOUNT_FREE_Y_SPACES_NEEDED.get(entityPose).iterator();
 
             while(g.hasNext()) {
                int i = g.next();
@@ -180,16 +180,16 @@
             }
          }
 
-         double e = this.getBoundingBox().maxY;
-         mutable.set((double)blockPos.getX(), e, (double)blockPos.getZ());
+         double e2 = this.getBoundingBox().maxY;
+         mutable.set((double)blockPos.getX(), e2, (double)blockPos.getZ());
          UnmodifiableIterator var22 = immutableList.iterator();
 
          while(var22.hasNext()) {
             EntityPose entityPose2 = (EntityPose)var22.next();
             double g = (double)passenger.getDimensions(entityPose2).height;
-            int j = MathHelper.ceil(e - (double)mutable.getY() + g);
+            int j = MathHelper.ceil(e2 - (double)mutable.getY() + g);
             double h = Dismounting.getCeilingHeight(mutable, j, (pos) -> this.world.getBlockState(pos).getCollisionShape(this.world, pos));
-            if (e + g <= h) {
+            if (e2 + g <= h) {
                passenger.setPose(entityPose2);
                break;
             }
