--- a/net/minecraft/entity/vehicle/BoatEntity.java	1970-01-12 08:58:04
+++ b/net/minecraft/entity/vehicle/BoatEntity.java	2023-03-30 20:22:55
@@ -4,6 +4,8 @@
 import com.google.common.collect.UnmodifiableIterator;
 import java.util.List;
 import java.util.function.IntFunction;
+import java.util.function.ToIntFunction;
+
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.Blocks;
@@ -890,7 +892,7 @@
       private final String name;
       private final Block baseBlock;
       public static final StringIdentifiable.Codec<BoatEntity.Type> CODEC = StringIdentifiable.createCodec(BoatEntity.Type::values);
-      private static final IntFunction<BoatEntity.Type> BY_ID = ValueLists.createIdToValueFunction(Enum::ordinal, values(), ValueLists.OutOfBoundsHandling.ZERO);
+      private static final IntFunction<BoatEntity.Type> BY_ID = ValueLists.createIdToValueFunction((ToIntFunction<Type>) Enum::ordinal, values(), ValueLists.OutOfBoundsHandling.ZERO);
 
       private Type(Block baseBlock, String name) {
          this.name = name;
