--- a/net/minecraft/loot/LootPool.java
+++ b/net/minecraft/loot/LootPool.java
@@ -118,7 +118,7 @@
       return new LootPool.Builder();
    }
 
-   public static class Builder implements LootFunctionConsumingBuilder<LootPool.Builder>, LootConditionConsumingBuilder<LootPool.Builder>, FabricLootPoolBuilder {
+   public static class Builder implements LootFunctionConsumingBuilder<LootPool.Builder>, LootConditionConsumingBuilder<LootPool.Builder> {
       private final List<LootPoolEntry> entries = Lists.newArrayList();
       private final List<LootCondition> conditions = Lists.newArrayList();
       private final List<LootFunction> functions = Lists.newArrayList();
@@ -147,6 +147,11 @@
       public LootPool.Builder conditionally(LootCondition.Builder builder) {
          this.conditions.add(builder.build());
          return this;
+      }
+
+      @Override
+      public Builder getThisConditionConsumingBuilder() {
+         return getThisFunctionConsumingBuilder();
       }
 
       public LootPool.Builder apply(LootFunction.Builder builder) {
