--- a/net/minecraft/loot/LootTableReporter.java
+++ b/net/minecraft/loot/LootTableReporter.java
@@ -54,12 +54,12 @@
    }
 
    public LootTableReporter withTable(String name, Identifier id) {
-      ImmutableSet<Identifier> immutableSet = ImmutableSet.builder().addAll(this.tables).add(id).build();
+      ImmutableSet<Identifier> immutableSet = ImmutableSet.<Identifier>builder().addAll(this.tables).add(id).build();
       return new LootTableReporter(this.messages, () -> this.getName() + name, this.contextType, this.conditionGetter, this.conditions, this.tableGetter, immutableSet);
    }
 
    public LootTableReporter withCondition(String name, Identifier id) {
-      ImmutableSet<Identifier> immutableSet = ImmutableSet.builder().addAll(this.conditions).add(id).build();
+      ImmutableSet<Identifier> immutableSet = ImmutableSet.<Identifier>builder().addAll(this.conditions).add(id).build();
       return new LootTableReporter(this.messages, () -> this.getName() + name, this.contextType, this.conditionGetter, immutableSet, this.tableGetter, this.tables);
    }
 
