--- a/java/net/minecraft/network/message/ArgumentSignatureDataMap.java
+++ b/java/net/minecraft/network/message/ArgumentSignatureDataMap.java
@@ -13,7 +13,7 @@
    private static final int MAX_ARGUMENT_NAME_LENGTH = 16;
 
    public ArgumentSignatureDataMap(PacketByteBuf buf) {
-      this(buf.readCollection(PacketByteBuf.getMaxValidator(ArrayList::new, 8), ArgumentSignatureDataMap.Entry::new));
+      this(buf.<Entry, List<Entry>>readCollection(PacketByteBuf.getMaxValidator(ArrayList::new, 8), Entry::new));
    }
 
    @Nullable
@@ -48,8 +48,6 @@
    }
 
    public static record Entry(String name, MessageSignatureData signature) {
-      final String name;
-      final MessageSignatureData signature;
 
       public Entry(PacketByteBuf buf) {
          this(buf.readString(16), MessageSignatureData.fromBuf(buf));
@@ -60,4 +58,4 @@
          MessageSignatureData.write(buf, this.signature);
       }
    }
 }
