--- a/net/minecraft/world/gen/feature/EndSpikeFeatureConfig.java
+++ b/net/minecraft/world/gen/feature/EndSpikeFeatureConfig.java
@@ -21,7 +21,7 @@
    private EndSpikeFeatureConfig(boolean crystalInvulnerable, List<EndSpikeFeature.Spike> spikes, Optional<BlockPos> crystalBeamTarget) {
       this.crystalInvulnerable = crystalInvulnerable;
       this.spikes = spikes;
-      this.crystalBeamTarget = (BlockPos)crystalBeamTarget.orElse((Object)null);
+      this.crystalBeamTarget = (BlockPos)crystalBeamTarget.orElse(null);
    }
 
    public boolean isCrystalInvulnerable() {
