--- a/net/minecraft/structure/StructureTemplate.java
+++ b/net/minecraft/structure/StructureTemplate.java
@@ -131,7 +131,7 @@
    }
 
    private static List<StructureTemplate.StructureBlockInfo> combineSorted(List<StructureTemplate.StructureBlockInfo> fullBlocks, List<StructureTemplate.StructureBlockInfo> blocksWithNbt, List<StructureTemplate.StructureBlockInfo> otherBlocks) {
-      Comparator<StructureTemplate.StructureBlockInfo> comparator = Comparator.comparingInt((blockInfo) -> blockInfo.pos.getY()).thenComparingInt((blockInfo) -> blockInfo.pos.getX()).thenComparingInt((blockInfo) -> blockInfo.pos.getZ());
+      Comparator<StructureTemplate.StructureBlockInfo> comparator = Comparator.<StructureTemplate.StructureBlockInfo>comparingInt((blockInfo) -> blockInfo.pos.getY()).thenComparingInt((blockInfo) -> blockInfo.pos.getX()).thenComparingInt((blockInfo) -> blockInfo.pos.getZ());
       fullBlocks.sort(comparator);
       otherBlocks.sort(comparator);
       blocksWithNbt.sort(comparator);
